# Les outils d'annotation

## Ensembl Variant Effect Predictor (VEP)

[VEP](https://www.ensembl.org/info/docs/tools/vep/index.html) est un outil d'annotation de variations développé en Perl sous licence Apache-2.0 au sein du projet Ensembl.  
Il permet d'informer quel(s) gène(s) et transcrits sont impactés par une variation et l'impact de cette variation sur ceux-ci mais également d'ajouter des annotations complémentaires inclues dans l'outil de base, récupérées de plugins ou directement d'une source d'informations personnelle sous format VCF, BED, GFF, GTF ou bigWig.

### Output

Par défaut son fichier de sortie est un fichier tabulé où toutes les annotations complémentaires sont ajoutées sous la forme clef=valeur dans une colonne `Extra`, mais il peut annoter directement un vcf ou produire un JSON.

??? abstract "Outputs avec annotation de base"

    === "Standard"

        **Commande**
        ```bash
        ./vep --input_file /opt/vep/.vep/inputs/sample.vcf --output_file /opt/vep/.vep/outputs/std_output --cache --offline --format vcf
        ```

        **Output**
        ```text
        ## ENSEMBL VARIANT EFFECT PREDICTOR v109.3
        ## Output produced at 2023-04-14 11:45:52
        ## Using cache in /opt/vep/.vep/homo_sapiens/109_GRCh37
        ## Using API version 109, DB version ?
        ## ensembl version 109.10baaec
        ## ensembl-variation version 109.18a12b6
        ## ensembl-funcgen version 109.cba2db8
        ## ensembl-io version 109.4946a86
        ## genebuild version 2011-04
        ## 1000genomes version phase3
        ## HGMD-PUBLIC version 20204
        ## dbSNP version 154
        ## COSMIC version 92
        ## ClinVar version 202012
        ## gencode version GENCODE 19
        ## gnomADe version r2.1
        ## sift version sift5.2.2
        ## assembly version GRCh37.p13
        ## regbuild version 1.0
        ## polyphen version 2.2.2
        ## Column descriptions:
        ## Uploaded_variation : Identifier of uploaded variant
        ## Location : Location of variant in standard coordinate format (chr:start or chr:start-end)
        ## Allele : The variant allele used to calculate the consequence
        ## Gene : Stable ID of affected gene
        ## Feature : Stable ID of feature
        ## Feature_type : Type of feature - Transcript, RegulatoryFeature or MotifFeature
        ## Consequence : Consequence type
        ## cDNA_position : Relative position of base pair in cDNA sequence
        ## CDS_position : Relative position of base pair in coding sequence
        ## Protein_position : Relative position of amino acid in protein
        ## Amino_acids : Reference and variant amino acids
        ## Codons : Reference and variant codon sequence
        ## Existing_variation : Identifier(s) of co-located known variants
        ## Extra column keys:
        ## IMPACT : Subjective impact classification of consequence type
        ## DISTANCE : Shortest distance from variant to transcript
        ## STRAND : Strand of the feature (1/-1)
        ## FLAGS : Transcript quality flags
        ## VEP command-line: vep --cache --database 0 --dir_plugins /plugins --format vcf --input_file [PATH]/sample.vcf --offline --output_file [PATH]/std_output
        #Uploaded_variation	Location	Allele	Gene	Feature	Feature_type	Consequence	cDNA_position	CDS_position	Protein_position	Amino_acids	Codons	Existing_variation	Extra
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000379370	Transcript	synonymous_variant	4211	4161	1387	T	acT/acC	-	IMPACT=LOW;STRAND=1
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000419249	Transcript	upstream_gene_variant	-	-	-	-	-	-	IMPACT=MODIFIER;DISTANCE=1634;STRAND=1;FLAGS=cds_start_NF,cds_end_NF
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000461111	Transcript	upstream_gene_variant	-	-	-	-	-	-	IMPACT=MODIFIER;DISTANCE=3064;STRAND=1
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000466223	Transcript	downstream_gene_variant	-	-	-	-	-	-	IMPACT=MODIFIER;DISTANCE=1057;STRAND=1
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000478677	Transcript	downstream_gene_variant	-	-	-	-	-	-	IMPACT=MODIFIER;DISTANCE=1084;STRAND=1
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000479707	Transcript	downstream_gene_variant	-	-	-	-	-	-	IMPACT=MODIFIER;DISTANCE=2573;STRAND=1
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000492947	Transcript	non_coding_transcript_exon_variant	394	-	-	-	-	-	IMPACT=MODIFIER;STRAND=1
        chr1_985266_C/T	chr1:985266	T	ENSG00000188157	ENST00000379370	Transcript	splice_polypyrimidine_tract_variant,intron_variant	-	-	-	-	-	-	IMPACT=LOW;STRAND=1
        chr1_985266_C/T	chr1:985266	T	ENSG00000188157	ENST00000419249	Transcript	upstream_gene_variant	-	-	-	-	-	-	IMPACT=MODIFIER;DISTANCE=670;STRAND=1;FLAGS=cds_start_NF,cds_end_NF
        ```


    === "VCF"

        **Commande**
        ```bash
        ./vep --input_file /opt/vep/.vep/inputs/sample.vcf --output_file /opt/vep/.vep/outputs/std_output.vcf --cache --offline --everything --format vcf --vcf
        ```

        **Note**

        - Si une annotation contient une `,` elle sera remplacée par une `&` dans le VCF produit.
        - Pour cet exemple, toutes les annotations disponibles de base dans VEP ont été appliquées.

        **Output**
        ```text
        ##fileformat=VCFv4.2
        ##FILTER=<ID=LowCoverage,Description="DP < 5">
        ##FILTER=<ID=SNP_filter,Description="DP < 10 || QD < 2.0 || FS > 60.0 || MQ < 40.0 || FBtypeScore > 13.0 || MappingQualityRankSum < -12.5 || ReadPosRankSum < -8.0">
        ##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Number of observation for each allele">
        ##FORMAT=<ID=AO,Number=A,Type=Integer,Description="Alternate allele observation count">
        ##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">
        ##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
        ##FORMAT=<ID=PL,Number=G,Type=Integer,Description="Normalized, Phred-scaled likelihoods for genotypes as defined in the VCF specification">
        ##FORMAT=<ID=QA,Number=A,Type=Integer,Description="Sum of quality of the alternate observations">
        ##FORMAT=<ID=QR,Number=1,Type=Integer,Description="Sum of quality of the reference observations">
        ##FORMAT=<ID=RO,Number=1,Type=Integer,Description="Reference allele observation count">
        ##INFO=<ID=AB,Number=A,Type=Float,Description="Allele balance at heterozygous sites: a number between 0 and 1 representing the ratio of reads showing the reference allele to all reads, considering only reads from individuals called as heterozygous">
        ##INFO=<ID=ABP,Number=A,Type=Float,Description="Allele balance probability at heterozygous sites: Phred-scaled upper-bounds estimate of the probability of observing the deviation between ABR and ABA given E(ABR/ABA) ~ 0.5, derived using Hoeffding's inequality">
        ##INFO=<ID=AC,Number=A,Type=Integer,Description="Allele count in genotypes, for each ALT allele, in the same order as listed">
        ##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency, for each ALT allele, in the same order as listed">
        ##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
        ##INFO=<ID=AO,Number=A,Type=Integer,Description="Count of full observations of this alternate haplotype.">
        ##INFO=<ID=CIGAR,Number=A,Type=String,Description="The extended CIGAR representation of each alternate allele, with the exception that '=' is replaced by 'M' to ease VCF parsing.  Note that INDEL alleles do not have the first matched base (which is provided by default, per the spec) referred to by the CIGAR.">
        ##INFO=<ID=DP,Number=1,Type=Integer,Description="Approximate read depth; some reads may have been filtered">
        ##VEP="v109" time="2023-04-14 12:14:01" cache="/opt/vep/.vep/homo_sapiens/109_GRCh37" ensembl-io=109.4946a86 ensembl=109.10baaec ensembl-funcgen=109.cba2db8 ensembl-variation=109.18a12b6 1000genomes="phase3" COSMIC="92" ClinVar="202012" HGMD-PUBLIC="20204" assembly="GRCh37.p13" dbSNP="154" gencode="GENCODE 19" genebuild="2011-04" gnomADe="r2.1" polyphen="2.2.2" regbuild="1.0" sift="sift5.2.2"
        ##INFO=<ID=CSQ,Number=.,Type=String,Description="Consequence annotations from Ensembl VEP. Format: Allele|Consequence|IMPACT|SYMBOL|Gene|Feature_type|Feature|BIOTYPE|EXON|INTRON|HGVSc|HGVSp|cDNA_position|CDS_position|Protein_position|Amino_acids|Codons|Existing_variation|DISTANCE|STRAND|FLAGS|VARIANT_CLASS|SYMBOL_SOURCE|HGNC_ID|CANONICAL|MANE_SELECT|MANE_PLUS_CLINICAL|TSL|APPRIS|CCDS|ENSP|SWISSPROT|TREMBL|UNIPARC|UNIPROT_ISOFORM|GENE_PHENO|SIFT|PolyPhen|DOMAINS|miRNA|HGVS_OFFSET|AF|AFR_AF|AMR_AF|EAS_AF|EUR_AF|SAS_AF|gnomADe_AF|gnomADe_AFR_AF|gnomADe_AMR_AF|gnomADe_ASJ_AF|gnomADe_EAS_AF|gnomADe_FIN_AF|gnomADe_NFE_AF|gnomADe_OTH_AF|gnomADe_SAS_AF|gnomADg_AF|gnomADg_AFR_AF|gnomADg_AMI_AF|gnomADg_AMR_AF|gnomADg_ASJ_AF|gnomADg_EAS_AF|gnomADg_FIN_AF|gnomADg_MID_AF|gnomADg_NFE_AF|gnomADg_OTH_AF|gnomADg_SAS_AF|MAX_AF|MAX_AF_POPS|CLIN_SIG|SOMATIC|PHENO|PUBMED|MOTIF_NAME|MOTIF_POS|HIGH_INF_POS|MOTIF_SCORE_CHANGE|TRANSCRIPTION_FACTORS">
        ##VEP-command-line='vep --cache --database 0 --dir_plugins /plugins --everything --format vcf --input_file [PATH]/sample.vcf --offline --output_file [PATH]/std_output.vcf --vcf'
        #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	22-23121-A-02-00	22-23123-A-02-00	22-24543-A-01-00
        chr1	984302	.	T	C	3877.41	PASS	AB=0.464088;ABP=5.03781;AC=4;AF=0.667;AN=6;AO=151;CIGAR=1X;DP=249;CSQ=C|synonymous_variant|LOW|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|24/36||ENST00000379370.2:c.4161T>C|ENSP00000368678.2:p.Thr1387%3D|4211|4161|1387|T|acT/acC|rs9442391||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1|||Gene3D:2.60.120.200&PROSITE_profiles:PS50025&PANTHER:PTHR10574&PANTHER:PTHR10574:SF227&Superfamily:SSF49899|||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&25741868|||||,C|upstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs9442391|1634|1|cds_start_NF&cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&25741868|||||,C|upstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron||||||||||rs9442391|3064|1||SNV|HGNC|329||||||||||||1||||||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&25741868|||||,C|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000466223|retained_intron||||||||||rs9442391|1057|1||SNV|HGNC|329||||||||||||1||||||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&25741868|||||,C|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000478677|retained_intron||||||||||rs9442391|1084|1||SNV|HGNC|329||||||||||||1||||||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&25741868|||||,C|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000479707|retained_intron||||||||||rs9442391|2573|1||SNV|HGNC|329||||||||||||1||||||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&25741868|||||,C|non_coding_transcript_exon_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000492947|retained_intron|1/2||ENST00000492947.1:n.394T>C||394|||||rs9442391||1||SNV|HGNC|329||||||||||||1||||||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&25741868|||||	GT:AD:AO:DP:PL:QA:QR:RO	1/1:1,67:67:68:1979,171,0:2234:34:1	0/1:52,35:35:87:812,0,1296:1190:1728:52	0/1:45,49:49:94:1219,0,1056:1666:1484:45
        chr1	985266	.	C	T	2801.75	PASS	AB=0.491667;ABP=3.08268;AC=4;AF=0.667;AN=6;AO=110;CIGAR=1X;DP=171;CSQ=T|splice_polypyrimidine_tract_variant&intron_variant|LOW|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding||26/35|ENST00000379370.2:c.4745-17C>T|||||||rs2275813||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||,T|upstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs2275813|670|1|cds_start_NF&cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||,T|upstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron||||||||||rs2275813|2100|1||SNV|HGNC|329||||||||||||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000466223|retained_intron||||||||||rs2275813|2021|1||SNV|HGNC|329||||||||||||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000478677|retained_intron||||||||||rs2275813|2048|1||SNV|HGNC|329||||||||||||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000479707|retained_intron||||||||||rs2275813|3537|1||SNV|HGNC|329||||||||||||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000492947|retained_intron||||||||||rs2275813|492|1||SNV|HGNC|329||||||||||||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,51:51:51:1549,154,0:1718:0:0	0/1:33,27:27:60:649,0,825:918:1114:33	0/1:28,32:32:60:782,0,679:1066:952:28
        chr1	986732	.	G	A	922.52	PASS	AB=0.415094;ABP=9.64763;AC=1;AF=0.167;AN=6;AO=44;CIGAR=1X;DP=356;CSQ=A|missense_variant|MODERATE|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|31/36||ENST00000379370.2:c.5353G>A|ENSP00000368678.2:p.Asp1785Asn|5403|5353|1785|D/N|Gac/Aac|rs144245019&COSV99062175||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1|tolerated(0.27)|benign(0)|Gene3D:2.60.120.200&Pfam:PF00054&PROSITE_profiles:PS50025&PANTHER:PTHR10574&PANTHER:PTHR10574:SF227&SMART:SM00282&Superfamily:SSF49899|||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&uncertain_significance|0&1|1&1|18414213&25741868|||||,A|upstream_gene_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic||||||||||rs144245019&COSV99062175|3681|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&uncertain_significance|0&1|1&1|18414213&25741868|||||,A|missense_variant|MODERATE|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding|4/9||ENST00000419249.1:c.262G>A|ENSP00000400771.1:p.Asp88Asn|260|262|88|D/N|Gac/Aac|rs144245019&COSV99062175||1|cds_start_NF&cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1|tolerated(0.48)|benign(0.006)|Gene3D:2.60.120.200&Pfam:PF00054&PROSITE_profiles:PS50025&PANTHER:PTHR10574&PANTHER:PTHR10574:SF227&Superfamily:SSF49899|||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&uncertain_significance|0&1|1&1|18414213&25741868|||||,A|upstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron||||||||||rs144245019&COSV99062175|634|1||SNV|HGNC|329||||||||||||1||||||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&uncertain_significance|0&1|1&1|18414213&25741868|||||,A|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000466223|retained_intron||||||||||rs144245019&COSV99062175|3487|1||SNV|HGNC|329||||||||||||1||||||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&uncertain_significance|0&1|1&1|18414213&25741868|||||,A|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000478677|retained_intron||||||||||rs144245019&COSV99062175|3514|1||SNV|HGNC|329||||||||||||1||||||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&uncertain_significance|0&1|1&1|18414213&25741868|||||,A|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000492947|retained_intron||||||||||rs144245019&COSV99062175|1958|1||SNV|HGNC|329||||||||||||1||||||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&uncertain_significance|0&1|1&1|18414213&25741868|||||,A|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00001771209|CTCF_binding_site||||||||||rs144245019&COSV99062175||||SNV||||||||||||||||||||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&uncertain_significance|0&1|1&1|18414213&25741868|||||	GT:AD:AO:DP:PL:QA:QR:RO	0/0:129,0:0:129:0,388,3908:0:4340:129	0/1:62,44:44:106:1030,0,1581:1496:2108:62	0/0:121,0:0:121:0,364,3683:0:4090:121
        chr1	987200	.	C	T	6412.21	PASS	AB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=219;CIGAR=1X;DP=219;CSQ=T|splice_donor_5th_base_variant&intron_variant|LOW|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding||33/35|ENST00000379370.2:c.5651+5C>T|||||||rs9803031||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&25741868|||||,T|upstream_gene_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic||||||||||rs9803031|3213|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&25741868|||||,T|splice_donor_5th_base_variant&intron_variant|LOW|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||6/8|ENST00000419249.1:c.558+5C>T|||||||rs9803031||1|cds_start_NF&cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&25741868|||||,T|upstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron||||||||||rs9803031|166|1||SNV|HGNC|329||||||||||||1||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000466223|retained_intron||||||||||rs9803031|3955|1||SNV|HGNC|329||||||||||||1||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000478677|retained_intron||||||||||rs9803031|3982|1||SNV|HGNC|329||||||||||||1||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000492947|retained_intron||||||||||rs9803031|2426|1||SNV|HGNC|329||||||||||||1||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&25741868|||||	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,63:63:63:1911,190,0:2120:0:0	1/1:0,66:66:66:1963,199,0:2178:0:0	1/1:0,90:90:90:2737,271,0:3038:0:0
        chr1	990280	.	C	T	4547.65	PASS	AB=0.446602;ABP=8.11221;AC=4;AF=0.667;AN=6;AO=175;CIGAR=1X;DP=289;CSQ=T|synonymous_variant|LOW|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|36/36||ENST00000379370.2:c.6057C>T|ENSP00000368678.2:p.Asp2019%3D|6107|6057|2019|D|gaC/gaT|rs4275402&COSV65070964||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1|||Gene3D:2.60.120.200&Pfam:PF00054&PROSITE_profiles:PS50025&PANTHER:PTHR10574&PANTHER:PTHR10574:SF227&SMART:SM00282&Superfamily:SSF49899|||0.5853|0.2617|0.7262|0.8214|0.6779|0.5838|0.6764|0.3114|0.7722|0.6575|0.8138|0.641|0.698|0.6758|0.6133||||||||||||0.8214|EAS|benign|0&1|1&1|18414213&25741868|||||,T|upstream_gene_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic||||||||||rs4275402&COSV65070964|133|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.5853|0.2617|0.7262|0.8214|0.6779|0.5838|0.6764|0.3114|0.7722|0.6575|0.8138|0.641|0.698|0.6758|0.6133||||||||||||0.8214|EAS|benign|0&1|1&1|18414213&25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs4275402&COSV65070964|950|1|cds_start_NF&cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.5853|0.2617|0.7262|0.8214|0.6779|0.5838|0.6764|0.3114|0.7722|0.6575|0.8138|0.641|0.698|0.6758|0.6133||||||||||||0.8214|EAS|benign|0&1|1&1|18414213&25741868|||||,T|upstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000427998|processed_transcript||||||||||rs4275402&COSV65070964|4834|1||SNV|Clone_based_vega_gene|||||||||||||||||||0.5853|0.2617|0.7262|0.8214|0.6779|0.5838|0.6764|0.3114|0.7722|0.6575|0.8138|0.641|0.698|0.6758|0.6133||||||||||||0.8214|EAS|benign|0&1|1&1|18414213&25741868|||||,T|non_coding_transcript_exon_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron|3/3||ENST00000461111.1:n.2173C>T||2173|||||rs4275402&COSV65070964||1||SNV|HGNC|329||||||||||||1||||||0.5853|0.2617|0.7262|0.8214|0.6779|0.5838|0.6764|0.3114|0.7722|0.6575|0.8138|0.641|0.698|0.6758|0.6133||||||||||||0.8214|EAS|benign|0&1|1&1|18414213&25741868|||||,T|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00001771211|CTCF_binding_site||||||||||rs4275402&COSV65070964||||SNV||||||||||||||||||||0.5853|0.2617|0.7262|0.8214|0.6779|0.5838|0.6764|0.3114|0.7722|0.6575|0.8138|0.641|0.698|0.6758|0.6133||||||||||||0.8214|EAS|benign|0&1|1&1|18414213&25741868|||||	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,83:83:83:2496,250,0:2770:0:0	0/1:56,39:39:95:910,0,1371:1326:1838:56	0/1:58,53:53:111:1283,0,1436:1794:1964:58
        chr1	990380	.	C	T	570.29	PASS	AB=0.55102;ABP=4.11819;AC=2;AF=0.333;AN=6;AO=27;CIGAR=1X;DP=63;CSQ=T|3_prime_UTR_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|36/36||ENST00000379370.2:c.*19C>T||6207|||||rs3121561&COSV65069616||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1||||||0.3434|0.5461|0.2594|0.1726|0.3091|0.3405|0.2913|0.5233|0.2137|0.3111|0.1794|0.3513|0.292|0.2986|0.3087||||||||||||0.5461|AFR|benign|0&1|1&1|25741868|||||,T|upstream_gene_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic||||||||||rs3121561&COSV65069616|33|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.3434|0.5461|0.2594|0.1726|0.3091|0.3405|0.2913|0.5233|0.2137|0.3111|0.1794|0.3513|0.292|0.2986|0.3087||||||||||||0.5461|AFR|benign|0&1|1&1|25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs3121561&COSV65069616|1050|1|cds_start_NF&cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.3434|0.5461|0.2594|0.1726|0.3091|0.3405|0.2913|0.5233|0.2137|0.3111|0.1794|0.3513|0.292|0.2986|0.3087||||||||||||0.5461|AFR|benign|0&1|1&1|25741868|||||,T|upstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000427998|processed_transcript||||||||||rs3121561&COSV65069616|4734|1||SNV|Clone_based_vega_gene|||||||||||||||||||0.3434|0.5461|0.2594|0.1726|0.3091|0.3405|0.2913|0.5233|0.2137|0.3111|0.1794|0.3513|0.292|0.2986|0.3087||||||||||||0.5461|AFR|benign|0&1|1&1|25741868|||||,T|non_coding_transcript_exon_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron|3/3||ENST00000461111.1:n.2273C>T||2273|||||rs3121561&COSV65069616||1||SNV|HGNC|329||||||||||||1||||||0.3434|0.5461|0.2594|0.1726|0.3091|0.3405|0.2913|0.5233|0.2137|0.3111|0.1794|0.3513|0.292|0.2986|0.3087||||||||||||0.5461|AFR|benign|0&1|1&1|25741868|||||,T|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00001771211|CTCF_binding_site||||||||||rs3121561&COSV65069616||||SNV||||||||||||||||||||0.3434|0.5461|0.2594|0.1726|0.3091|0.3405|0.2913|0.5233|0.2137|0.3111|0.1794|0.3513|0.292|0.2986|0.3087||||||||||||0.5461|AFR|benign|0&1|1&1|25741868|||||	GT:AD:AO:DP:PL:QA:QR:RO	0/0:14,0:0:14:0,42,432:0:476:14	0/1:8,14:14:22:365,0,182:476:272:8	0/1:14,13:13:27:320,0,343:442:468:14
        chr1	990417	.	T	C	328.36	PASS	AB=0.580645;ABP=4.76149;AC=2;AF=0.333;AN=6;AO=18;CIGAR=1X;DP=41;CSQ=C|3_prime_UTR_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|36/36||ENST00000379370.2:c.*56T>C||6244|||||rs2465136&COSV65069776||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1||||||0.3840|0.6475|0.2723|0.1786|0.3211|0.3834|||||||||||||||||||||0.6475|AFR||0&1|0&1||||||,C|non_coding_transcript_exon_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic|1/2||ENST00000418300.1:n.5T>C||5|||||rs2465136&COSV65069776||1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.3840|0.6475|0.2723|0.1786|0.3211|0.3834|||||||||||||||||||||0.6475|AFR||0&1|0&1||||||,C|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs2465136&COSV65069776|1087|1|cds_start_NF&cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.3840|0.6475|0.2723|0.1786|0.3211|0.3834|||||||||||||||||||||0.6475|AFR||0&1|0&1||||||,C|upstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000427998|processed_transcript||||||||||rs2465136&COSV65069776|4697|1||SNV|Clone_based_vega_gene|||||||||||||||||||0.3840|0.6475|0.2723|0.1786|0.3211|0.3834|||||||||||||||||||||0.6475|AFR||0&1|0&1||||||,C|non_coding_transcript_exon_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron|3/3||ENST00000461111.1:n.2310T>C||2310|||||rs2465136&COSV65069776||1||SNV|HGNC|329||||||||||||1||||||0.3840|0.6475|0.2723|0.1786|0.3211|0.3834|||||||||||||||||||||0.6475|AFR||0&1|0&1||||||	GT:AD:AO:DP:PL:QA:QR:RO	0/0:10,0:0:10:0,30,309:0:340:10	0/1:5,9:9:14:237,0,114:306:170:5	0/1:8,9:9:17:220,0,197:298:272:8
        chr1	990517	.	C	T	182.67	SNP_filter	AB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=8;CIGAR=1X;DP=8;CSQ=T|3_prime_UTR_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|36/36||ENST00000379370.2:c.*156C>T||6344|||||rs2710872||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1||||||0.9179|0.8419|0.928|0.999|0.8877|0.9611|||||||||||||||||||||0.999|EAS|||1|32314116|||||,T|non_coding_transcript_exon_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic|1/2||ENST00000418300.1:n.105C>T||105|||||rs2710872||1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.9179|0.8419|0.928|0.999|0.8877|0.9611|||||||||||||||||||||0.999|EAS|||1|32314116|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs2710872|1187|1|cds_start_NF&cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.9179|0.8419|0.928|0.999|0.8877|0.9611|||||||||||||||||||||0.999|EAS|||1|32314116|||||,T|upstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000427998|processed_transcript||||||||||rs2710872|4597|1||SNV|Clone_based_vega_gene|||||||||||||||||||0.9179|0.8419|0.928|0.999|0.8877|0.9611|||||||||||||||||||||0.999|EAS|||1|32314116|||||,T|non_coding_transcript_exon_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron|3/3||ENST00000461111.1:n.2410C>T||2410|||||rs2710872||1||SNV|HGNC|329||||||||||||1||||||0.9179|0.8419|0.928|0.999|0.8877|0.9611|||||||||||||||||||||0.999|EAS|||1|32314116|||||	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,1:1:1:34,3,0:34:0:0	1/1:0,4:4:4:126,12,0:136:0:0	1/1:0,3:3:3:95,9,0:102:0:0
        chr1	990984	.	G	A	82.39	LowCoverage;SNP_filter	AB=0;ABP=0;AC=2;AF=1.00;AN=2;AO=3;CIGAR=1X;DP=3;CSQ=A|3_prime_UTR_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|36/36||ENST00000379370.2:c.*623G>A||6811|||||rs8014||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1||||||0.9223|0.8555|0.9323|0.999|0.8887|0.9611|||||||||||||||||||||0.999|EAS|||||||||,A|intron_variant&non_coding_transcript_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic||1/1|ENST00000418300.1:n.184-294G>A|||||||rs8014||1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.9223|0.8555|0.9323|0.999|0.8887|0.9611|||||||||||||||||||||0.999|EAS|||||||||,A|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs8014|1654|1|cds_start_NF&cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.9223|0.8555|0.9323|0.999|0.8887|0.9611|||||||||||||||||||||0.999|EAS|||||||||,A|upstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000427998|processed_transcript||||||||||rs8014|4130|1||SNV|Clone_based_vega_gene|||||||||||||||||||0.9223|0.8555|0.9323|0.999|0.8887|0.9611|||||||||||||||||||||0.999|EAS|||||||||,A|non_coding_transcript_exon_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron|3/3||ENST00000461111.1:n.2877G>A||2877|||||rs8014||1||SNV|HGNC|329||||||||||||1||||||0.9223|0.8555|0.9323|0.999|0.8887|0.9611|||||||||||||||||||||0.999|EAS|||||||||	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,3:3:3:95,9,0:102:0:0	.	.
        chr1	1007203	.	A	G	3110.06	PASS	AB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=105;CIGAR=1X;DP=105;CSQ=G|downstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000412397|transcribed_unprocessed_pseudogene||||||||||rs4633229|2468|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.8413|0.8797|0.8112|0.9196|0.7883|0.7843|0.816|0.8594|0.8533|0.8069|0.9078|0.8538|0.7842|0.8041|0.7851||||||||||||0.9196|EAS|||||||||,G|synonymous_variant|LOW|RNF223|ENSG00000237330|Transcript|ENST00000453464|protein_coding|2/2||ENST00000453464.2:c.744T>C|ENSP00000410533.1:p.Asp248%3D|1045|744|248|D|gaT/gaC|rs4633229||-1||SNV|HGNC|40020|YES|||||CCDS53257.1|ENSP00000410533|RN223_HUMAN||UPI0001A5E6EF|||||PANTHER:PTHR22791&PANTHER:PTHR22791:SF4|||0.8413|0.8797|0.8112|0.9196|0.7883|0.7843|0.816|0.8594|0.8533|0.8069|0.9078|0.8538|0.7842|0.8041|0.7851||||||||||||0.9196|EAS|||||||||	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,26:26:26:752,78,0:832:0:0	1/1:0,46:46:46:1411,138,0:1564:0:0	1/1:0,33:33:33:1013,99,0:1122:0:0
        chr1	1007222	.	G	T	1780.81	PASS	AB=0.685185;ABP=35.1803;AC=2;AF=0.333;AN=6;AO=74;CIGAR=1X;DP=143;CSQ=T|downstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000412397|transcribed_unprocessed_pseudogene||||||||||rs71628928&COSV69251322|2487|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.1174|0.0257|0.0951|0.1091|0.163|0.2188|0.1412|0.02294|0.07825|0.1934|0.07649|0.2412|0.148|0.1606|0.2119||||||||||||0.2412|gnomADe_FIN||0&1|0&1||||||,T|missense_variant|MODERATE|RNF223|ENSG00000237330|Transcript|ENST00000453464|protein_coding|2/2||ENST00000453464.2:c.725C>A|ENSP00000410533.1:p.Pro242His|1026|725|242|P/H|cCc/cAc|rs71628928&COSV69251322||-1||SNV|HGNC|40020|YES|||||CCDS53257.1|ENSP00000410533|RN223_HUMAN||UPI0001A5E6EF|||tolerated(0.22)|possibly_damaging(0.897)|Low_complexity_(Seg):seg&PANTHER:PTHR22791&PANTHER:PTHR22791:SF4|||0.1174|0.0257|0.0951|0.1091|0.163|0.2188|0.1412|0.02294|0.07825|0.1934|0.07649|0.2412|0.148|0.1606|0.2119||||||||||||0.2412|gnomADe_FIN||0&1|0&1||||||	GT:AD:AO:DP:PL:QA:QR:RO	0/0:35,0:0:35:0,105,1034:0:1146:35	0/1:16,40:40:56:1044,0,277:1344:492:16	0/1:18,34:34:52:887,0,397:1156:612:18
        chr1	1007432	.	G	A	3295.23	PASS	AB=0.44375;ABP=7.40753;AC=4;AF=0.667;AN=6;AO=128;CIGAR=1X;DP=217;CSQ=A|downstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000412397|transcribed_unprocessed_pseudogene||||||||||rs4333796&COSV69251112|2697|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.5321|0.2307|0.6643|0.7917|0.5616|0.5481|0.6458|0.3916|0.7639|0.6074|0.8456|0.5287|0.6098|0.6154|0.5811||||||||||||0.8456|gnomADe_EAS||0&1|0&1||||||,A|missense_variant|MODERATE|RNF223|ENSG00000237330|Transcript|ENST00000453464|protein_coding|2/2||ENST00000453464.2:c.515C>T|ENSP00000410533.1:p.Ala172Val|816|515|172|A/V|gCg/gTg|rs4333796&COSV69251112||-1||SNV|HGNC|40020|YES|||||CCDS53257.1|ENSP00000410533|RN223_HUMAN||UPI0001A5E6EF|||tolerated(0.7)|benign(0)|Low_complexity_(Seg):seg&PANTHER:PTHR22791&PANTHER:PTHR22791:SF4|||0.5321|0.2307|0.6643|0.7917|0.5616|0.5481|0.6458|0.3916|0.7639|0.6074|0.8456|0.5287|0.6098|0.6154|0.5811||||||||||||0.8456|gnomADe_EAS||0&1|0&1||||||	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,57:57:57:1740,172,0:1930:0:0	0/1:51,36:36:87:823,0,1302:1202:1734:51	0/1:38,35:35:73:854,0,946:1190:1292:38

        ```

    === "Tabulé"

        **Commande**
        ```bash
        ./vep --input_file /opt/vep/.vep/inputs/sample.vcf --output_file /opt/vep/.vep/outputs/std_output.vcf --cache --offline --format vcf --vcf
        ```

        **Output**
        ```text
        ## ENSEMBL VARIANT EFFECT PREDICTOR v109.3
        ## Output produced at 2023-04-14 11:45:13
        ## Using cache in /opt/vep/.vep/homo_sapiens/109_GRCh37
        ## Using API version 109, DB version ?
        ## ensembl-funcgen version 109.cba2db8
        ## ensembl-variation version 109.18a12b6
        ## ensembl-io version 109.4946a86
        ## ensembl version 109.10baaec
        ## 1000genomes version phase3
        ## polyphen version 2.2.2
        ## dbSNP version 154
        ## assembly version GRCh37.p13
        ## COSMIC version 92
        ## genebuild version 2011-04
        ## gnomADe version r2.1
        ## regbuild version 1.0
        ## sift version sift5.2.2
        ## HGMD-PUBLIC version 20204
        ## gencode version GENCODE 19
        ## ClinVar version 202012
        ## Column descriptions:
        ## Uploaded_variation : Identifier of uploaded variant
        ## Location : Location of variant in standard coordinate format (chr:start or chr:start-end)
        ## Allele : The variant allele used to calculate the consequence
        ## Gene : Stable ID of affected gene
        ## Feature : Stable ID of feature
        ## Feature_type : Type of feature - Transcript, RegulatoryFeature or MotifFeature
        ## Consequence : Consequence type
        ## cDNA_position : Relative position of base pair in cDNA sequence
        ## CDS_position : Relative position of base pair in coding sequence
        ## Protein_position : Relative position of amino acid in protein
        ## Amino_acids : Reference and variant amino acids
        ## Codons : Reference and variant codon sequence
        ## Existing_variation : Identifier(s) of co-located known variants
        ## IMPACT : Subjective impact classification of consequence type
        ## DISTANCE : Shortest distance from variant to transcript
        ## STRAND : Strand of the feature (1/-1)
        ## FLAGS : Transcript quality flags
        ## VEP command-line: vep --cache --database 0 --dir_plugins /plugins --format vcf --input_file [PATH]/sample.vcf --offline --output_file [PATH]/std_output.csv --tab
        #Uploaded_variation	Location	Allele	Gene	Feature	Feature_type	Consequence	cDNA_position	CDS_position	Protein_position	Amino_acids	Codons	Existing_variation	IMPACT	DISTANCE	STRAND	FLAGS
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000379370	Transcript	synonymous_variant	4211	4161	1387	T	acT/acC	-	LOW	-	1	-
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000419249	Transcript	upstream_gene_variant	-	-	-	-	-	-	MODIFIER	1634	1	cds_start_NF,cds_end_NF
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000461111	Transcript	upstream_gene_variant	-	-	-	-	-	-	MODIFIER	3064	1	-
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000466223	Transcript	downstream_gene_variant	-	-	-	-	-	-	MODIFIER	1057	1	-
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000478677	Transcript	downstream_gene_variant	-	-	-	-	-	-	MODIFIER	1084	1	-
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000479707	Transcript	downstream_gene_variant	-	-	-	-	-	-	MODIFIER	2573	1	-
        chr1_984302_T/C	chr1:984302	C	ENSG00000188157	ENST00000492947	Transcript	non_coding_transcript_exon_variant	394	-	-	-	-	-	MODIFIER	-	1	-
        chr1_985266_C/T	chr1:985266	T	ENSG00000188157	ENST00000379370	Transcript	splice_polypyrimidine_tract_variant,intron_variant	-	-	-	-	-	-	LOW	-	1	-
        chr1_985266_C/T	chr1:985266	T	ENSG00000188157	ENST00000419249	Transcript	upstream_gene_variant	-	-	-	-	-	-	MODIFIER	670	1	cds_start_NF,cds_end_NF
        chr1_985266_C/T	chr1:985266	T	ENSG00000188157	ENST00000461111	Transcript	upstream_gene_variant	-	-	-	-	-	-	MODIFIER	2100	1	-

        ```

    === "JSON"

        **Commande**
        ```bash
        ./vep --input_file /opt/vep/.vep/inputs/sample.vcf --output_file /opt/vep/.vep/outputs/std_output.json --cache --offline --format vcf --json
        ```

        **Output**
        ```json
        {"allele_string":"T/C","most_severe_consequence":"synonymous_variant","id":".","transcript_consequences":[{"variant_allele":"C","gene_id":"ENSG00000188157","impact":"LOW","cds_end":4161,"transcript_id":"ENST00000379370","strand":1,"amino_acids":"T","cds_start":4161,"cdna_end":4211,"codons":"acT/acC","protein_end":1387,"cdna_start":4211,"protein_start":1387,"consequence_terms":["synonymous_variant"]},{"impact":"MODIFIER","gene_id":"ENSG00000188157","variant_allele":"C","flags":["cds_start_NF","cds_end_NF"],"strand":1,"transcript_id":"ENST00000419249","distance":1634,"consequence_terms":["upstream_gene_variant"]},{"transcript_id":"ENST00000461111","strand":1,"impact":"MODIFIER","variant_allele":"C","gene_id":"ENSG00000188157","consequence_terms":["upstream_gene_variant"],"distance":3064},{"distance":1057,"consequence_terms":["downstream_gene_variant"],"variant_allele":"C","gene_id":"ENSG00000188157","impact":"MODIFIER","transcript_id":"ENST00000466223","strand":1},{"distance":1084,"consequence_terms":["downstream_gene_variant"],"impact":"MODIFIER","variant_allele":"C","gene_id":"ENSG00000188157","transcript_id":"ENST00000478677","strand":1},{"consequence_terms":["downstream_gene_variant"],"distance":2573,"transcript_id":"ENST00000479707","strand":1,"impact":"MODIFIER","variant_allele":"C","gene_id":"ENSG00000188157"},{"cdna_end":394,"cdna_start":394,"consequence_terms":["non_coding_transcript_exon_variant"],"variant_allele":"C","gene_id":"ENSG00000188157","impact":"MODIFIER","transcript_id":"ENST00000492947","strand":1}],"end":984302,"input":"chr1\t984302\t.\tT\tC\t3877.41\tPASS\tAB=0.464088;ABP=5.03781;AC=4;AF=0.667;AN=6;AO=151;CIGAR=1X;DP=249\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:1,67:67:68:1979,171,0:2234:34:1\t0/1:52,35:35:87:812,0,1296:1190:1728:52\t0/1:45,49:49:94:1219,0,1056:1666:1484:45","seq_region_name":"chr1","start":984302,"strand":1,"assembly_name":"GRCh37"}
        {"seq_region_name":"chr1","input":"chr1\t985266\t.\tC\tT\t2801.75\tPASS\tAB=0.491667;ABP=3.08268;AC=4;AF=0.667;AN=6;AO=110;CIGAR=1X;DP=171\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,51:51:51:1549,154,0:1718:0:0\t0/1:33,27:27:60:649,0,825:918:1114:33\t0/1:28,32:32:60:782,0,679:1066:952:28","end":985266,"transcript_consequences":[{"strand":1,"consequence_terms":["splice_polypyrimidine_tract_variant","intron_variant"],"transcript_id":"ENST00000379370","impact":"LOW","gene_id":"ENSG00000188157","variant_allele":"T"},{"strand":1,"transcript_id":"ENST00000419249","gene_id":"ENSG00000188157","variant_allele":"T","flags":["cds_start_NF","cds_end_NF"],"impact":"MODIFIER","consequence_terms":["upstream_gene_variant"],"distance":670},{"distance":2100,"consequence_terms":["upstream_gene_variant"],"gene_id":"ENSG00000188157","variant_allele":"T","impact":"MODIFIER","strand":1,"transcript_id":"ENST00000461111"},{"distance":2021,"consequence_terms":["downstream_gene_variant"],"impact":"MODIFIER","gene_id":"ENSG00000188157","variant_allele":"T","strand":1,"transcript_id":"ENST00000466223"},{"transcript_id":"ENST00000478677","strand":1,"impact":"MODIFIER","variant_allele":"T","gene_id":"ENSG00000188157","consequence_terms":["downstream_gene_variant"],"distance":2048},{"distance":3537,"consequence_terms":["downstream_gene_variant"],"gene_id":"ENSG00000188157","variant_allele":"T","impact":"MODIFIER","strand":1,"transcript_id":"ENST00000479707"},{"strand":1,"transcript_id":"ENST00000492947","gene_id":"ENSG00000188157","variant_allele":"T","impact":"MODIFIER","consequence_terms":["downstream_gene_variant"],"distance":492}],"id":".","most_severe_consequence":"splice_polypyrimidine_tract_variant","allele_string":"C/T","assembly_name":"GRCh37","strand":1,"start":985266}
        {"start":986732,"strand":1,"assembly_name":"GRCh37","id":".","transcript_consequences":[{"protein_start":1785,"consequence_terms":["missense_variant"],"cdna_start":5403,"protein_end":1785,"codons":"Gac/Aac","cdna_end":5403,"transcript_id":"ENST00000379370","amino_acids":"D/N","strand":1,"cds_start":5353,"impact":"MODERATE","cds_end":5353,"variant_allele":"A","gene_id":"ENSG00000188157"},{"distance":3681,"consequence_terms":["upstream_gene_variant"],"impact":"MODIFIER","gene_id":"ENSG00000242590","variant_allele":"A","strand":1,"transcript_id":"ENST00000418300"},{"consequence_terms":["missense_variant"],"cdna_start":260,"protein_end":88,"transcript_id":"ENST00000419249","cds_end":262,"gene_id":"ENSG00000188157","variant_allele":"A","flags":["cds_start_NF","cds_end_NF"],"protein_start":88,"codons":"Gac/Aac","cdna_end":260,"amino_acids":"D/N","strand":1,"cds_start":262,"impact":"MODERATE"},{"strand":1,"transcript_id":"ENST00000461111","gene_id":"ENSG00000188157","variant_allele":"A","impact":"MODIFIER","consequence_terms":["upstream_gene_variant"],"distance":634},{"distance":3487,"consequence_terms":["downstream_gene_variant"],"impact":"MODIFIER","gene_id":"ENSG00000188157","variant_allele":"A","strand":1,"transcript_id":"ENST00000466223"},{"impact":"MODIFIER","variant_allele":"A","gene_id":"ENSG00000188157","transcript_id":"ENST00000478677","strand":1,"distance":3514,"consequence_terms":["downstream_gene_variant"]},{"consequence_terms":["downstream_gene_variant"],"distance":1958,"strand":1,"transcript_id":"ENST00000492947","gene_id":"ENSG00000188157","variant_allele":"A","impact":"MODIFIER"}],"most_severe_consequence":"missense_variant","allele_string":"G/A","seq_region_name":"chr1","input":"chr1\t986732\t.\tG\tA\t922.52\tPASS\tAB=0.415094;ABP=9.64763;AC=1;AF=0.167;AN=6;AO=44;CIGAR=1X;DP=356\tGT:AD:AO:DP:PL:QA:QR:RO\t0/0:129,0:0:129:0,388,3908:0:4340:129\t0/1:62,44:44:106:1030,0,1581:1496:2108:62\t0/0:121,0:0:121:0,364,3683:0:4090:121","end":986732}
        {"assembly_name":"GRCh37","start":987200,"strand":1,"seq_region_name":"chr1","end":987200,"input":"chr1\t987200\t.\tC\tT\t6412.21\tPASS\tAB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=219;CIGAR=1X;DP=219\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,63:63:63:1911,190,0:2120:0:0\t1/1:0,66:66:66:1963,199,0:2178:0:0\t1/1:0,90:90:90:2737,271,0:3038:0:0","id":".","transcript_consequences":[{"gene_id":"ENSG00000188157","variant_allele":"T","impact":"LOW","consequence_terms":["splice_donor_5th_base_variant","intron_variant"],"strand":1,"transcript_id":"ENST00000379370"},{"transcript_id":"ENST00000418300","strand":1,"impact":"MODIFIER","variant_allele":"T","gene_id":"ENSG00000242590","consequence_terms":["upstream_gene_variant"],"distance":3213},{"transcript_id":"ENST00000419249","strand":1,"variant_allele":"T","flags":["cds_start_NF","cds_end_NF"],"gene_id":"ENSG00000188157","impact":"LOW","consequence_terms":["splice_donor_5th_base_variant","intron_variant"]},{"distance":166,"consequence_terms":["upstream_gene_variant"],"impact":"MODIFIER","gene_id":"ENSG00000188157","variant_allele":"T","strand":1,"transcript_id":"ENST00000461111"},{"consequence_terms":["downstream_gene_variant"],"distance":3955,"strand":1,"transcript_id":"ENST00000466223","gene_id":"ENSG00000188157","variant_allele":"T","impact":"MODIFIER"},{"distance":3982,"consequence_terms":["downstream_gene_variant"],"impact":"MODIFIER","gene_id":"ENSG00000188157","variant_allele":"T","strand":1,"transcript_id":"ENST00000478677"},{"consequence_terms":["downstream_gene_variant"],"distance":2426,"transcript_id":"ENST00000492947","strand":1,"variant_allele":"T","gene_id":"ENSG00000188157","impact":"MODIFIER"}],"allele_string":"C/T","most_severe_consequence":"splice_donor_5th_base_variant"}
        {"end":990280,"input":"chr1\t990280\t.\tC\tT\t4547.65\tPASS\tAB=0.446602;ABP=8.11221;AC=4;AF=0.667;AN=6;AO=175;CIGAR=1X;DP=289\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,83:83:83:2496,250,0:2770:0:0\t0/1:56,39:39:95:910,0,1371:1326:1838:56\t0/1:58,53:53:111:1283,0,1436:1794:1964:58","seq_region_name":"chr1","most_severe_consequence":"synonymous_variant","allele_string":"C/T","id":".","transcript_consequences":[{"cdna_end":6107,"codons":"gaC/gaT","protein_end":2019,"cdna_start":6107,"protein_start":2019,"consequence_terms":["synonymous_variant"],"variant_allele":"T","gene_id":"ENSG00000188157","impact":"LOW","cds_end":6057,"transcript_id":"ENST00000379370","cds_start":6057,"strand":1,"amino_acids":"D"},{"strand":1,"transcript_id":"ENST00000418300","gene_id":"ENSG00000242590","variant_allele":"T","impact":"MODIFIER","consequence_terms":["upstream_gene_variant"],"distance":133},{"gene_id":"ENSG00000188157","variant_allele":"T","flags":["cds_start_NF","cds_end_NF"],"impact":"MODIFIER","strand":1,"transcript_id":"ENST00000419249","distance":950,"consequence_terms":["downstream_gene_variant"]},{"distance":4834,"consequence_terms":["upstream_gene_variant"],"gene_id":"ENSG00000217801","variant_allele":"T","impact":"MODIFIER","strand":1,"transcript_id":"ENST00000427998"},{"cdna_end":2173,"cdna_start":2173,"consequence_terms":["non_coding_transcript_exon_variant"],"gene_id":"ENSG00000188157","variant_allele":"T","impact":"MODIFIER","strand":1,"transcript_id":"ENST00000461111"}],"assembly_name":"GRCh37","start":990280,"strand":1}
        {"strand":1,"start":990380,"assembly_name":"GRCh37","transcript_consequences":[{"transcript_id":"ENST00000379370","strand":1,"impact":"MODIFIER","variant_allele":"T","gene_id":"ENSG00000188157","consequence_terms":["3_prime_UTR_variant"],"cdna_start":6207,"cdna_end":6207},{"impact":"MODIFIER","gene_id":"ENSG00000242590","variant_allele":"T","strand":1,"transcript_id":"ENST00000418300","distance":33,"consequence_terms":["upstream_gene_variant"]},{"impact":"MODIFIER","gene_id":"ENSG00000188157","variant_allele":"T","flags":["cds_start_NF","cds_end_NF"],"strand":1,"transcript_id":"ENST00000419249","distance":1050,"consequence_terms":["downstream_gene_variant"]},{"transcript_id":"ENST00000427998","strand":1,"variant_allele":"T","gene_id":"ENSG00000217801","impact":"MODIFIER","consequence_terms":["upstream_gene_variant"],"distance":4734},{"cdna_start":2273,"consequence_terms":["non_coding_transcript_exon_variant"],"cdna_end":2273,"strand":1,"transcript_id":"ENST00000461111","gene_id":"ENSG00000188157","variant_allele":"T","impact":"MODIFIER"}],"id":".","allele_string":"C/T","most_severe_consequence":"3_prime_UTR_variant","seq_region_name":"chr1","end":990380,"input":"chr1\t990380\t.\tC\tT\t570.29\tPASS\tAB=0.55102;ABP=4.11819;AC=2;AF=0.333;AN=6;AO=27;CIGAR=1X;DP=63\tGT:AD:AO:DP:PL:QA:QR:RO\t0/0:14,0:0:14:0,42,432:0:476:14\t0/1:8,14:14:22:365,0,182:476:272:8\t0/1:14,13:13:27:320,0,343:442:468:14"}
        {"input":"chr1\t990417\t.\tT\tC\t328.36\tPASS\tAB=0.580645;ABP=4.76149;AC=2;AF=0.333;AN=6;AO=18;CIGAR=1X;DP=41\tGT:AD:AO:DP:PL:QA:QR:RO\t0/0:10,0:0:10:0,30,309:0:340:10\t0/1:5,9:9:14:237,0,114:306:170:5\t0/1:8,9:9:17:220,0,197:298:272:8","end":990417,"seq_region_name":"chr1","allele_string":"T/C","most_severe_consequence":"3_prime_UTR_variant","id":".","transcript_consequences":[{"cdna_start":6244,"consequence_terms":["3_prime_UTR_variant"],"cdna_end":6244,"transcript_id":"ENST00000379370","strand":1,"variant_allele":"C","gene_id":"ENSG00000188157","impact":"MODIFIER"},{"cdna_end":5,"cdna_start":5,"consequence_terms":["non_coding_transcript_exon_variant"],"variant_allele":"C","gene_id":"ENSG00000242590","impact":"MODIFIER","transcript_id":"ENST00000418300","strand":1},{"gene_id":"ENSG00000188157","variant_allele":"C","flags":["cds_start_NF","cds_end_NF"],"impact":"MODIFIER","strand":1,"transcript_id":"ENST00000419249","distance":1087,"consequence_terms":["downstream_gene_variant"]},{"consequence_terms":["upstream_gene_variant"],"distance":4697,"transcript_id":"ENST00000427998","strand":1,"impact":"MODIFIER","variant_allele":"C","gene_id":"ENSG00000217801"},{"cdna_end":2310,"cdna_start":2310,"consequence_terms":["non_coding_transcript_exon_variant"],"variant_allele":"C","gene_id":"ENSG00000188157","impact":"MODIFIER","transcript_id":"ENST00000461111","strand":1}],"assembly_name":"GRCh37","start":990417,"strand":1}
        {"most_severe_consequence":"3_prime_UTR_variant","allele_string":"C/T","transcript_consequences":[{"cdna_start":6344,"consequence_terms":["3_prime_UTR_variant"],"cdna_end":6344,"transcript_id":"ENST00000379370","strand":1,"variant_allele":"T","gene_id":"ENSG00000188157","impact":"MODIFIER"},{"consequence_terms":["non_coding_transcript_exon_variant"],"cdna_start":105,"cdna_end":105,"transcript_id":"ENST00000418300","strand":1,"impact":"MODIFIER","variant_allele":"T","gene_id":"ENSG00000242590"},{"distance":1187,"consequence_terms":["downstream_gene_variant"],"impact":"MODIFIER","flags":["cds_start_NF","cds_end_NF"],"variant_allele":"T","gene_id":"ENSG00000188157","transcript_id":"ENST00000419249","strand":1},{"strand":1,"transcript_id":"ENST00000427998","gene_id":"ENSG00000217801","variant_allele":"T","impact":"MODIFIER","consequence_terms":["upstream_gene_variant"],"distance":4597},{"cdna_end":2410,"cdna_start":2410,"consequence_terms":["non_coding_transcript_exon_variant"],"gene_id":"ENSG00000188157","variant_allele":"T","impact":"MODIFIER","strand":1,"transcript_id":"ENST00000461111"}],"id":".","input":"chr1\t990517\t.\tC\tT\t182.67\tSNP_filter\tAB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=8;CIGAR=1X;DP=8\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,1:1:1:34,3,0:34:0:0\t1/1:0,4:4:4:126,12,0:136:0:0\t1/1:0,3:3:3:95,9,0:102:0:0","end":990517,"seq_region_name":"chr1","strand":1,"start":990517,"assembly_name":"GRCh37"}
        {"seq_region_name":"chr1","input":"chr1\t990984\t.\tG\tA\t82.39\tLowCoverage;SNP_filter\tAB=0;ABP=0;AC=2;AF=1.00;AN=2;AO=3;CIGAR=1X;DP=3\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,3:3:3:95,9,0:102:0:0\t.\t.","end":990984,"transcript_consequences":[{"consequence_terms":["3_prime_UTR_variant"],"cdna_start":6811,"cdna_end":6811,"strand":1,"transcript_id":"ENST00000379370","impact":"MODIFIER","gene_id":"ENSG00000188157","variant_allele":"A"},{"impact":"MODIFIER","gene_id":"ENSG00000242590","variant_allele":"A","consequence_terms":["intron_variant","non_coding_transcript_variant"],"strand":1,"transcript_id":"ENST00000418300"},{"impact":"MODIFIER","variant_allele":"A","flags":["cds_start_NF","cds_end_NF"],"gene_id":"ENSG00000188157","transcript_id":"ENST00000419249","strand":1,"distance":1654,"consequence_terms":["downstream_gene_variant"]},{"gene_id":"ENSG00000217801","variant_allele":"A","impact":"MODIFIER","strand":1,"transcript_id":"ENST00000427998","distance":4130,"consequence_terms":["upstream_gene_variant"]},{"consequence_terms":["non_coding_transcript_exon_variant"],"cdna_start":2877,"cdna_end":2877,"strand":1,"transcript_id":"ENST00000461111","impact":"MODIFIER","gene_id":"ENSG00000188157","variant_allele":"A"}],"id":".","most_severe_consequence":"3_prime_UTR_variant","allele_string":"G/A","assembly_name":"GRCh37","strand":1,"start":990984}
        {"assembly_name":"GRCh37","strand":1,"start":1007203,"input":"chr1\t1007203\t.\tA\tG\t3110.06\tPASS\tAB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=105;CIGAR=1X;DP=105\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,26:26:26:752,78,0:832:0:0\t1/1:0,46:46:46:1411,138,0:1564:0:0\t1/1:0,33:33:33:1013,99,0:1122:0:0","end":1007203,"seq_region_name":"chr1","allele_string":"A/G","most_severe_consequence":"synonymous_variant","transcript_consequences":[{"impact":"MODIFIER","variant_allele":"G","gene_id":"ENSG00000217801","transcript_id":"ENST00000412397","strand":1,"distance":2468,"consequence_terms":["downstream_gene_variant"]},{"cds_start":744,"amino_acids":"D","strand":-1,"transcript_id":"ENST00000453464","cds_end":744,"impact":"LOW","gene_id":"ENSG00000237330","variant_allele":"G","protein_start":248,"consequence_terms":["synonymous_variant"],"cdna_start":1045,"codons":"gaT/gaC","protein_end":248,"cdna_end":1045}],"id":"."}
        {"assembly_name":"GRCh37","strand":1,"start":1007222,"seq_region_name":"chr1","end":1007222,"input":"chr1\t1007222\t.\tG\tT\t1780.81\tPASS\tAB=0.685185;ABP=35.1803;AC=2;AF=0.333;AN=6;AO=74;CIGAR=1X;DP=143\tGT:AD:AO:DP:PL:QA:QR:RO\t0/0:35,0:0:35:0,105,1034:0:1146:35\t0/1:16,40:40:56:1044,0,277:1344:492:16\t0/1:18,34:34:52:887,0,397:1156:612:18","transcript_consequences":[{"distance":2487,"consequence_terms":["downstream_gene_variant"],"impact":"MODIFIER","variant_allele":"T","gene_id":"ENSG00000217801","transcript_id":"ENST00000412397","strand":1},{"cdna_end":1026,"protein_end":242,"codons":"cCc/cAc","cdna_start":1026,"protein_start":242,"consequence_terms":["missense_variant"],"gene_id":"ENSG00000237330","variant_allele":"T","cds_end":725,"impact":"MODERATE","cds_start":725,"amino_acids":"P/H","strand":-1,"transcript_id":"ENST00000453464"}],"id":".","allele_string":"G/T","most_severe_consequence":"missense_variant"}
        {"strand":1,"start":1007432,"assembly_name":"GRCh37","most_severe_consequence":"missense_variant","allele_string":"G/A","transcript_consequences":[{"distance":2697,"consequence_terms":["downstream_gene_variant"],"impact":"MODIFIER","gene_id":"ENSG00000217801","variant_allele":"A","strand":1,"transcript_id":"ENST00000412397"},{"cds_end":515,"impact":"MODERATE","gene_id":"ENSG00000237330","variant_allele":"A","strand":-1,"cds_start":515,"amino_acids":"A/V","transcript_id":"ENST00000453464","codons":"gCg/gTg","protein_end":172,"cdna_end":816,"consequence_terms":["missense_variant"],"protein_start":172,"cdna_start":816}],"id":".","input":"chr1\t1007432\t.\tG\tA\t3295.23\tPASS\tAB=0.44375;ABP=7.40753;AC=4;AF=0.667;AN=6;AO=128;CIGAR=1X;DP=217\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,57:57:57:1740,172,0:1930:0:0\t0/1:51,36:36:87:823,0,1302:1202:1734:51\t0/1:38,35:35:73:854,0,946:1190:1292:38","end":1007432,"seq_region_name":"chr1"}
        ```

### Annotations avancées

Il est possible de compléter les annotations de base de VEP par des annotations issues de bases de données prévues à cet effet (plugins) ou créées par l'utilisateur (custom).

#### Plugins

Les [plugins](https://www.ensembl.org/info/docs/tools/vep/script/vep_plugins.html) sont des extensions de VEP nécessitant une installation (simplement avec la commande `perl INSTALL.pl -a p --PLUGINS dbNSFP,CADD`). Une fois l'installation faite et la source d'annotations récupérée, il suffit pour ajouter ces informations d'appeler le plugin lors du lancement d'une annotation, préciser le chemin de la source d'annotation et selon les plugins, préciser les colonnes désirées.

Par exemple : 

``` hl_lines="2"
./vep --input_file /opt/vep/.vep/inputs/sample.vcf --output_file /opt/vep/.vep/outputs/std_output.vcf --cache --offline --everything --format vcf --vcf \
--plugin dbNSFP,path/to/file.gz,CADD_raw_hg19,REVEL_score,Polyphen2_HVAR_score
```

#### Custom

Il est également possible d'ajouter des annotations de fichiers BED, VCF, GFF, GTF ou bigWig tant qu'ils ont été préalablement indexés via [tabix](https://www.htslib.org/doc/tabix.html).  
Il suffit ensuite de préciser au lancement d'une annotation le chemin vers le fichier, le nom pour nommer les colonnes, le format de fichier, le type d'annotation, s'il faut rapporter les coordonnées de la feature recouvrante puis les champs à récupérer.

Par exemple :

``` hl_lines="2"
./vep --input_file /opt/vep/.vep/inputs/sample.vcf --output_file /opt/vep/.vep/outputs/std_output.vcf --cache --offline --everything --format vcf --vcf \
--custom [PATH]/clinvar_20230326.vcf.gz,ClinVar,vcf,exact,0,CLNDN,CLNSIG,CLNREVSTAT
```

### Précision sur l'annotation de VCF

!!! note
    Il est préférable de bien connaître les spécificités du [format VCF](../vcf)

Voici un exemple de VCF réunissant des colonnes de plugins et de la base ClinVar la plus récente, mais n'affichant que le transcrit considéré prioritaire pour plus de lisibilité.

??? abstract "Exemple de fichier VCF annoté par VEP"

    ``` hl_lines="3 4 27"
    [...]
    ##INFO=<ID=DP,Number=1,Type=Integer,Description="Approximate read depth; some reads may have been filtered">
    ##VEP="v109" time="2023-04-12 19:40:52" cache="/opt/vep/.vep/homo_sapiens/109_GRCh37" ensembl-funcgen=109.cba2db8 ensembl-variation=109.18a12b6 ensembl-io=109.4946a86 ensembl=109.10baaec 1000genomes="phase3" COSMIC="92" ClinVar="202012" HGMD-PUBLIC="20204" assembly="GRCh37.p13" dbSNP="154" gencode="GENCODE 19" genebuild="2011-04" gnomADe="r2.1" polyphen="2.2.2" regbuild="1.0" sift="sift5.2.2"
    ##INFO=<ID=CSQ,Number=.,Type=String,Description="Consequence annotations from Ensembl VEP. Format: Gene|Feature|Feature_Type|HGVSg|HGVSc|HGVSp|Consequence|IMPACT|VARIANT_CLASS|Codons|Amino_acids|cDNA_position|CDS_position|ClinVar_CLNDN|ClinVar_CLNSIG|ClinVar_CLNREVSTAT|LOEUF|CADD_raw_hg19|GERP++_RS|REVEL_score|Polyphen2_HVAR|gnomAD_genomes_AC|gnomAD_genomes_AF|gnomAD_genomes_nhomalt|gnomAD_genomes_POPMAX_AF|SpliceAI_pred_DP_AG|SpliceAI_pred_DP_AL|SpliceAI_pred_DP_DG|SpliceAI_pred_DP_DL|SpliceAI_pred_DS_AG|SpliceAI_pred_DS_AL|SpliceAI_pred_DS_DG|SpliceAI_pred_DS_DL">
    ##CADD_raw_hg19=(from dbNSFP4.3a_grch37) CADD raw score for functional prediction of a SNP using the hg19 model. Please refer to Kircher et al. (2014) Nature Genetics 46(3):310-5 for details. The larger the score the more likely the SNP has damaging effect. Scores range from -6.458163 to 18.301497 in dbNSFP. Please note the following copyright statement for CADD: "CADD scores (http://cadd.gs.washington.edu/) are Copyright 2013 University of Washington and Hudson-Alpha Institute for Biotechnology (all rights reserved) but are freely available for all academic, non-commercial applications. For commercial licensing information contact Jennifer McCullar (mccullaj@uw.edu)."
    ##GERP++_RS=(from dbNSFP4.3a_grch37) GERP++ RS score, the larger the score, the more conserved the site. Scores range from -12.3 to 6.17.
    ##Polyphen2_HVAR_score=(from dbNSFP4.3a_grch37) Polyphen2 score based on HumVar, i.e. hvar_prob. The score ranges from 0 to 1. Multiple entries separated by ";", corresponding to Uniprot_acc.
    ##REVEL_score=(from dbNSFP4.3a_grch37) REVEL is an ensemble score based on 13 individual scores for predicting the pathogenicity of missense variants. Scores range from 0 to 1. The larger the score the more likely the SNP has damaging effect. "REVEL scores are freely available for non-commercial use. For other uses, please contact Weiva Sieh" (weiva.sieh@mssm.edu) Multiple entries are separated by ";", corresponding to Ensembl_transcriptid.
    ##gnomAD_genomes_AC=(from dbNSFP4.3a_grch37) Alternative allele count in the whole gnomAD genome samples v3.1 For mtDNA, this is sum of AC_hom ("Allele count restricted to variants with a heteroplasmy level >= 0.95") and AC_het ("Allele count restricted to variants with a heteroplasmy level >= 0.10 and < 0.95")
    ##gnomAD_genomes_AF=(from dbNSFP4.3a_grch37) Alternative allele frequency in the whole gnomAD genome samples v3.1 For mtDNA, this is sum of AF_hom ("Allele frequency restricted to variants with a heteroplasmy level >= 0.95") and AF_het ("Allele frequency restricted to variants with a heteroplasmy level >= 0.10 and < 0.95")
    ##gnomAD_genomes_POPMAX_AF=(from dbNSFP4.3a_grch37) Maximum allele frequency across populations (excluding samples of Ashkenazi, Finnish, and indeterminate ancestry)
    ##gnomAD_genomes_nhomalt=(from dbNSFP4.3a_grch37) Count of individuals with homozygous alternative allele in the whole gnomAD genome samples v3.1 For mtDNA, this is AC_hom ("Allele count restricted to variants with a heteroplasmy level >= 0.95")
    ##SpliceAI_pred_DP_AG=SpliceAI predicted effect on splicing. Delta position for acceptor gain
    ##SpliceAI_pred_DP_AL=SpliceAI predicted effect on splicing. Delta position for acceptor loss
    ##SpliceAI_pred_DP_DG=SpliceAI predicted effect on splicing. Delta position for donor gain
    ##SpliceAI_pred_DP_DL=SpliceAI predicted effect on splicing. Delta position for donor loss
    ##SpliceAI_pred_DS_AG=SpliceAI predicted effect on splicing. Delta score for acceptor gain
    ##SpliceAI_pred_DS_AL=SpliceAI predicted effect on splicing. Delta score for acceptor loss
    ##SpliceAI_pred_DS_DG=SpliceAI predicted effect on splicing. Delta score for donor gain
    ##SpliceAI_pred_DS_DL=SpliceAI predicted effect on splicing. Delta score for donor loss
    ##SpliceAI_pred_SYMBOL=SpliceAI gene symbol
    ##LOEUF=Loss-of-function observed/expected upper bound fraction
    ##INFO=<ID=ClinVar,Number=.,Type=String,Description="/opt/vep/.vep/hg19/clinvar/clinvar_20230326.vcf.gz">
    ##INFO=<ID=ClinVar_CLNDN,Number=.,Type=String,Description="CLNDN field from /opt/vep/.vep/hg19/clinvar/clinvar_20230326.vcf.gz">
    ##INFO=<ID=ClinVar_CLNSIG,Number=.,Type=String,Description="CLNSIG field from /opt/vep/.vep/hg19/clinvar/clinvar_20230326.vcf.gz">
    ##INFO=<ID=ClinVar_CLNREVSTAT,Number=.,Type=String,Description="CLNREVSTAT field from /opt/vep/.vep/hg19/clinvar/clinvar_20230326.vcf.gz">
    ##VEP-command-line='vep --cache --custom [PATH]/clinvar_20230326.vcf.gz,ClinVar,vcf,exact,0,CLNDN,CLNSIG,CLNREVSTAT --database 0 --dir_plugins /plugins --fields Gene,Feature,Feature_Type,HGVSg,HGVSc,HGVSp,Consequence,IMPACT,VARIANT_CLASS,Codons,Amino_acids,cDNA_position,CDS_position,ClinVar_CLNDN,ClinVar_CLNSIG,ClinVar_CLNREVSTAT,LOEUF,CADD_raw_hg19,GERP++_RS,REVEL_score,Polyphen2_HVAR,gnomAD_genomes_AC,gnomAD_genomes_AF,gnomAD_genomes_nhomalt,gnomAD_genomes_POPMAX_AF,SpliceAI_pred_DP_AG,SpliceAI_pred_DP_AL,SpliceAI_pred_DP_DG,SpliceAI_pred_DP_DL,SpliceAI_pred_DS_AG,SpliceAI_pred_DS_AL,SpliceAI_pred_DS_DG,SpliceAI_pred_DS_DL --force_overwrite --fork 6 --format vcf --hgvs --input_file [PATH]/sample.vcf --no_stats --offline --output_file [PATH]/example_pick.vcf --pick --plugin [PATH]/loeuf_dataset.tsv.gz,match_by=transcript --vcf'
    #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	22-23121-A-02-00	22-23123-A-02-00	22-24543-A-01-00
    chr1	984302	.	T	C	3877.41	PASS	AB=0.464088;ABP=5.03781;AC=4;AF=0.667;AN=6;AO=151;CIGAR=1X;DP=249;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.4161T>C|ENSP00000368678.2:p.Thr1387%3D|synonymous_variant|LOW||acT/acC|T|4211|4161|Congenital_myasthenic_syndrome_8&not_specified|Benign|criteria_provided&_multiple_submitters&_no_conflicts|4.3500e-01|||||||||-26|18|0|-2|0.00|0.00|0.00|0.00	GT:AD:AO:DP:PL:QA:QR:RO	1/1:1,67:67:68:1979,171,0:2234:34:1	0/1:52,35:35:87:812,0,1296:1190:1728:52	0/1:45,49:49:94:1219,0,1056:1666:1484:45
    chr1	985266	.	C	T	2801.75	PASS	AB=0.491667;ABP=3.08268;AC=4;AF=0.667;AN=6;AO=110;CIGAR=1X;DP=171;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.4745-17C>T||splice_polypyrimidine_tract_variant&intron_variant|LOW||||||Congenital_myasthenic_syndrome_8&not_specified|Benign|criteria_provided&_multiple_submitters&_no_conflicts|4.3500e-01|||||||||17|-4|25|-11|0.14|0.00|0.00|0.00	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,51:51:51:1549,154,0:1718:0:0	0/1:33,27:27:60:649,0,825:918:1114:33	0/1:28,32:32:60:782,0,679:1066:952:28
    chr1	986732	.	G	A	922.52	PASS	AB=0.415094;ABP=9.64763;AC=1;AF=0.167;AN=6;AO=44;CIGAR=1X;DP=356;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.5353G>A|ENSP00000368678.2:p.Asp1785Asn|missense_variant|MODERATE||Gac/Aac|D/N|5403|5353|Congenital_myasthenic_syndrome_8&not_specified&not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&_conflicting_interpretations|4.3500e-01|0.256992|-2.07|0.069&.||1031|6.785660e-03|1|1.145360e-02|3|-37|3|17|0.00|0.00|0.00|0.00	GT:AD:AO:DP:PL:QA:QR:RO	0/0:129,0:0:129:0,388,3908:0:4340:129	0/1:62,44:44:106:1030,0,1581:1496:2108:62	0/0:121,0:0:121:0,364,3683:0:4090:121
    chr1	987200	.	C	T	6412.21	PASS	AB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=219;CIGAR=1X;DP=219;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.5651+5C>T||splice_donor_5th_base_variant&intron_variant|LOW||||||Congenital_myasthenic_syndrome_8&not_specified|Benign|criteria_provided&_multiple_submitters&_no_conflicts|4.3500e-01|||||||||-27|-5|-5|-29|0.00|0.00|0.00|0.00	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,63:63:63:1911,190,0:2120:0:0	1/1:0,66:66:66:1963,199,0:2178:0:0	1/1:0,90:90:90:2737,271,0:3038:0:0
    chr1	990280	.	C	T	4547.65	PASS	AB=0.446602;ABP=8.11221;AC=4;AF=0.667;AN=6;AO=175;CIGAR=1X;DP=289;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.6057C>T|ENSP00000368678.2:p.Asp2019%3D|synonymous_variant|LOW||gaC/gaT|D|6107|6057|Congenital_myasthenic_syndrome_8&not_specified|Benign|criteria_provided&_multiple_submitters&_no_conflicts|4.3500e-01|||||||||37|2|-23|-4|0.00|0.00|0.00|0.00	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,83:83:83:2496,250,0:2770:0:0	0/1:56,39:39:95:910,0,1371:1326:1838:56	0/1:58,53:53:111:1283,0,1436:1794:1964:58
    chr1	990380	.	C	T	570.29	PASS	AB=0.55102;ABP=4.11819;AC=2;AF=0.333;AN=6;AO=27;CIGAR=1X;DP=63;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.*19C>T||3_prime_UTR_variant|MODIFIER||||6207||not_specified|Benign|criteria_provided&_multiple_submitters&_no_conflicts|4.3500e-01|||||||||7|-7|7|26|0.00|0.00|0.00|0.00	GT:AD:AO:DP:PL:QA:QR:RO	0/0:14,0:0:14:0,42,432:0:476:14	0/1:8,14:14:22:365,0,182:476:272:8	0/1:14,13:13:27:320,0,343:442:468:14
    chr1	990417	.	T	C	328.36	PASS	AB=0.580645;ABP=4.76149;AC=2;AF=0.333;AN=6;AO=18;CIGAR=1X;DP=41;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.*56T>C||3_prime_UTR_variant|MODIFIER||||6244||not_provided|Benign|criteria_provided&_single_submitter|4.3500e-01|||||||||-44|29|8|-30|0.00|0.00|0.00|0.00	GT:AD:AO:DP:PL:QA:QR:RO	0/0:10,0:0:10:0,30,309:0:340:10	0/1:5,9:9:14:237,0,114:306:170:5	0/1:8,9:9:17:220,0,197:298:272:8
    chr1	990517	.	C	T	182.67	SNP_filter	AB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=8;CIGAR=1X;DP=8;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.*156C>T||3_prime_UTR_variant|MODIFIER||||6344||Congenital_myasthenic_syndrome_8&not_provided|Benign|criteria_provided&_multiple_submitters&_no_conflicts|4.3500e-01|||||||||14|-28|-5|18|0.00|0.05|0.00|0.00	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,1:1:1:34,3,0:34:0:0	1/1:0,4:4:4:126,12,0:136:0:0	1/1:0,3:3:3:95,9,0:102:0:0
    chr1	990984	.	G	A	82.39	LowCoverage;SNP_filter	AB=0;ABP=0;AC=2;AF=1.00;AN=2;AO=3;CIGAR=1X;DP=3;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.*623G>A||3_prime_UTR_variant|MODIFIER||||6811|||||4.3500e-01|||||||||5|-25|-42|5|0.00|0.00|0.00|0.00	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,3:3:3:95,9,0:102:0:0	.	.
    chr1	1007203	.	A	G	3110.06	PASS	AB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=105;CIGAR=1X;DP=105;CSQ=ENSG00000237330|ENST00000453464|||ENST00000453464.2:c.744T>C|ENSP00000410533.1:p.Asp248%3D|synonymous_variant|LOW||gaT/gaC|D|1045|744||||1.8670e+00|||||||||-7|-46|-26|2|0.00|0.00|0.01|0.00	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,26:26:26:752,78,0:832:0:0	1/1:0,46:46:46:1411,138,0:1564:0:0	1/1:0,33:33:33:1013,99,0:1122:0:0
    chr1	1007222	.	G	T	1780.81	PASS	AB=0.685185;ABP=35.1803;AC=2;AF=0.333;AN=6;AO=74;CIGAR=1X;DP=143;CSQ=ENSG00000237330|ENST00000453464|||ENST00000453464.2:c.725C>A|ENSP00000410533.1:p.Pro242His|missense_variant|MODERATE||cCc/cAc|P/H|1026|725||||1.8670e+00|1.523231|3.44|0.182||18137|1.192080e-01|1398|2.038880e-01|-26|-35|-45|16|0.00|0.00|0.00|0.00	GT:AD:AO:DP:PL:QA:QR:RO	0/0:35,0:0:35:0,105,1034:0:1146:35	0/1:16,40:40:56:1044,0,277:1344:492:16	0/1:18,34:34:52:887,0,397:1156:612:18
    chr1	1007432	.	G	A	3295.23	PASS	AB=0.44375;ABP=7.40753;AC=4;AF=0.667;AN=6;AO=128;CIGAR=1X;DP=217;CSQ=ENSG00000237330|ENST00000453464|||ENST00000453464.2:c.515C>T|ENSP00000410533.1:p.Ala172Val|missense_variant|MODERATE||gCg/gTg|A/V|816|515||||1.8670e+00|-0.654559|-7.35|0.132||75853|4.996380e-01|20418|7.826930e-01|48|16|28|44|0.00|0.00|0.00|0.00	GT:AD:AO:DP:PL:QA:QR:RO	1/1:0,57:57:57:1740,172,0:1930:0:0	0/1:51,36:36:87:823,0,1302:1202:1734:51	0/1:38,35:35:73:854,0,946:1190:1292:38
    ```

#### Le header

Lorsqu'il est utilisé, VEP ajoute dans le header

- la version du script et des DB utilisées
- le champs `##INFO=<ID=CSQ[...]>` précisant l'ordre des annotations ajoutées pour chaque transcrit
- La définition de toutes les colonnes personnalisées utilisées
- la ligne de commande utilisée pour lancer l'annotation (peut être incomplète ou obfusquée dans le cas d'un recours à un fichier `config`)

#### Le champ CSQ

L'annotation même est ajoutée dans la colonne INFO via un champs "CSQ" (par défaut) dédié.

- Pour chaque transcrit relevé, les annotations sont insérées dans l'ordre spécifié par la ligne `##INFO=<ID=CSQ[...]>` du header et séparées par des `|`.  
- Les transcrits quant à eux sont séparés par des virgules `,`  
- Enfin, le CSQ se conclue par un point-virgule `;` ou une tabulation.

##### Exemple de plusieurs transcrits

Exemple d'une annotation sur plusieurs transcrits (le retour à la ligne a été ajouté pour une meilleure lisibilité)

```text
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	22-23121-A-02-00	22-23123-A-02-00	22-24543-A-01-00
chr1	986732	.	G	A	922.52	PASS	AB=0.415094;ABP=9.64763;AC=1;AF=0.167;AN=6;AO=44;CIGAR=1X;DP=356; \
    CSQ=AGRN|ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.5353G>A|ENSP00000368678.2:p.Asp1785Asn|missense_variant|MODERATE||Gac/Aac|D/N|5403|5353|Congenital_myasthenic_syndrome_8&not_specified&not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&_conflicting_interpretations|4.3500e-01|0.256992|-2.07|0.069&.||1031|6.785660e-03|1|1.145360e-02|3|-37|3|17|0.00|0.00|0.00|0.00|1
        ,RP11-54O7.14|ENSG00000242590|ENST00000418300|||||upstream_gene_variant|MODIFIER||||||Congenital_myasthenic_syndrome_8&not_specified&not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&_conflicting_interpretations||||||||||||||||||
        ,AGRN|ENSG00000188157|ENST00000419249|||ENST00000419249.1:c.262G>A|ENSP00000400771.1:p.Asp88Asn|missense_variant|MODERATE||Gac/Aac|D/N|260|262|Congenital_myasthenic_syndrome_8&not_specified&not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&_conflicting_interpretations|6.7500e-01|0.256992|-2.07|0.069&.||1031|6.785660e-03|1|1.145360e-02|3|-37|3|17|0.00|0.00|0.00|0.00|
        ,AGRN|ENSG00000188157|ENST00000461111|||||upstream_gene_variant|MODIFIER||||||Congenital_myasthenic_syndrome_8&not_specified&not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&_conflicting_interpretations||||||||||3|-37|3|17|0.00|0.00|0.00|0.00|
        ,AGRN|ENSG00000188157|ENST00000466223|||||downstream_gene_variant|MODIFIER||||||Congenital_myasthenic_syndrome_8&not_specified&not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&_conflicting_interpretations||||||||||3|-37|3|17|0.00|0.00|0.00|0.00|
        ,AGRN|ENSG00000188157|ENST00000478677|||||downstream_gene_variant|MODIFIER||||||Congenital_myasthenic_syndrome_8&not_specified&not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&_conflicting_interpretations||||||||||3|-37|3|17|0.00|0.00|0.00|0.00|
        ,AGRN|ENSG00000188157|ENST00000492947|||||downstream_gene_variant|MODIFIER||||||Congenital_myasthenic_syndrome_8&not_specified&not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&_conflicting_interpretations||||||||||3|-37|3|17|0.00|0.00|0.00|0.00|	GT:AD:AO:DP:PL:QA:QR:RO	0/0:129,0:0:129:0,388,3908:0:4340:129	0/1:62,44:44:106:1030,0,1581:1496:2108:62	0/0:121,0:0:121:0,364,3683:0:4090:121

```

Il est à noter que le deuxième transcrit annoté concerne un second gène (*upstream_gene_variant* signifiant que la variation est en amont de ce gène).

##### Exemple de plusieurs ALT

Dans le cas d'un VCF non normalisé, plusieurs ALT sont notées sur une seule ligne du VCF. VEP fera l'annotation pour toutes les ALT au sein du même champ CSQ et les séparera par des virgules. Il reste possible de les différencier grâce à la colonne `Allele` permettant de savoir quelle ALT est concernée par l'annotation jusqu'à la virgule suivante.

```text
// CSQ = Allele|Consequence|Impact|Symbol|Gene|Feature_type|Feature|…
// Pour les variations : 
#CHROM	POS	ID	REF	ALT
chr1	889455	.	G	A, T [...]
CSQ=A|stop_gained|HIGH|NOC2L|ENSG00000188976|transcript|ENST00000327044|protein_coding|7/19|c.706C>T|p.Gln236*|756/2790|706/2250|236/749||
    ,T|missense_variant|MODERATE|NOC2L|ENSG00000188976|transcript|ENST00000327044|protein_coding|7/19|c.706C>A|p.Gln236Lys|756/2790|706/2250|236/749||
    ,A|downstream_gene_variant|MODIFIER|NOC2L|ENSG00000188976|transcript|ENST00000487214|processed_transcript||n.*865C>T|||||351|
    ,T|downstream_gene_variant|MODIFIER|NOC2L|ENSG00000188976|transcript|ENST00000487214|processed_transcript||n.*865C>A|||||351|
    ,A|downstream_gene_variant|MODIFIER|NOC2L|ENSG00000188976|transcript|ENST00000469563|retained_intron||n.*878C>T|||||4171|
    ,T|downstream_gene_variant|MODIFIER|NOC2L|ENSG00000188976|transcript|ENST00000469563|retained_intron||n.*878C>A|||||4171|
    ,A|non_coding_exon_variant|MODIFIER|NOC2L|ENSG00000188976|transcript|ENST00000477976|retained_intron|5/17|n.2153C>T||||||
    ,T|non_coding_exon_variant|MODIFIER|NOC2L|ENSG00000188976|transcript|ENST00000477976|retained_intron|5/17|n.2153C>A||||||;
```

##### Reformatage par VEP

Si une annotation contient une virgule ou un pipe, VEP la remplace par une esperluette `&`

Par exemple :

```text
// Dans le VCF de ClinVar
[...]CLNDN=Congenital_myasthenic_syndrome_8|not_specified|not_provided;CLNHGVS=NC_000001.10:g.986732G>A;CLNREVSTAT=criteria_provided,_conflicting_interpretations;CLNSIG=Conflicting_interpretations_of_pathogenicity[...]
// Dans le VCF annoté
[...]|Congenital_myasthenic_syndrome_8&not_specified&not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&_conflicting_interpretations|[...]
```
