
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
        <meta name="author" content="DiaghoProject">
      
      
        <link rel="canonical" href="https://docs.diagho.com/opensource_docs/bioinformatics/annotation_tools/">
      
      
        <link rel="prev" href="../vcf/">
      
      
        <link rel="next" href="../databases/">
      
      
      <link rel="icon" href="../../../images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.4.2">
    
    
      
        <title>Les outils d'annotation - Diagho Documentation</title>
      
    
    
      <link rel="stylesheet" href="../../../assets/stylesheets/main.d451bc0e.min.css">
      
        
        <link rel="stylesheet" href="../../../assets/stylesheets/palette.a5377069.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="deep-purple" data-md-color-accent="orange">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#les-outils-dannotation" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../../.." title="Diagho Documentation" class="md-header__button md-logo" aria-label="Diagho Documentation" data-md-component="logo">
      
  <img src="../../../images/favicon.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Diagho Documentation
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Les outils d'annotation
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/DiaghoProject/docs" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    DiaghoProject/docs
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../../.." title="Diagho Documentation" class="md-nav__button md-logo" aria-label="Diagho Documentation" data-md-component="logo">
      
  <img src="../../../images/favicon.png" alt="logo">

    </a>
    Diagho Documentation
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/DiaghoProject/docs" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    DiaghoProject/docs
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
    <li class="md-nav__item">
      <a href="../../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
    
    
      
        
          
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          
          <div class="md-nav__link md-nav__container">
            <a href="../../" class="md-nav__link ">
              
  
  <span class="md-ellipsis">
    Opensource Documentation
  </span>
  

            </a>
            
              
              <label class="md-nav__link " for="__nav_2" id="__nav_2_label" tabindex="0">
                <span class="md-nav__icon md-icon"></span>
              </label>
            
          </div>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Opensource Documentation
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../sitemap/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Organisation du site
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    
    
      
        
          
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_3" >
        
          
          
          <div class="md-nav__link md-nav__container">
            <a href="../../back-end/" class="md-nav__link ">
              
  
  <span class="md-ellipsis">
    Back end
  </span>
  

            </a>
            
              
              <label class="md-nav__link " for="__nav_2_3" id="__nav_2_3_label" tabindex="0">
                <span class="md-nav__icon md-icon"></span>
              </label>
            
          </div>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_3">
            <span class="md-nav__icon md-icon"></span>
            Back end
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../back-end/CI-CD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    CI-CD
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../back-end/architecture/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Diagho Architecture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../back-end/diagho-core/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    diagho-core
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../back-end/formats/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    API Endpoint standard formats
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../back-end/main_models/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Principaux modèles utilisés
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../back-end/passwords/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Bonnes pratiques de gestion des mots de passe
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../back-end/project_user_stories/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    projects
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../back-end/sequencings/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    sequencings
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../back-end/strategie_analyses/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Strategie Analyses
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../back-end/treatment_vcf/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Treatment of vcf files
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    
    
      
        
          
        
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_3_12" >
        
          
          
          <div class="md-nav__link md-nav__container">
            <a href="../../back-end/variant_analyses/" class="md-nav__link ">
              
  
  <span class="md-ellipsis">
    Variant analyses
  </span>
  

            </a>
            
              
              <label class="md-nav__link " for="__nav_2_3_12" id="__nav_2_3_12_label" tabindex="0">
                <span class="md-nav__icon md-icon"></span>
              </label>
            
          </div>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_2_3_12_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_3_12">
            <span class="md-nav__icon md-icon"></span>
            Variant analyses
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../back-end/variant_analyses/STEPS/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Steps
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
    
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_4" checked>
        
          
          <label class="md-nav__link" for="__nav_2_4" id="__nav_2_4_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Bioinformatics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_4_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2_4">
            <span class="md-nav__icon md-icon"></span>
            Bioinformatics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../vcf/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Le format VCF
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Les outils d'annotation
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Les outils d'annotation
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#ensembl-variant-effect-predictor-vep" class="md-nav__link">
    Ensembl Variant Effect Predictor (VEP)
  </a>
  
    <nav class="md-nav" aria-label="Ensembl Variant Effect Predictor (VEP)">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#output" class="md-nav__link">
    Output
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#annotations-avancees" class="md-nav__link">
    Annotations avancées
  </a>
  
    <nav class="md-nav" aria-label="Annotations avancées">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#plugins" class="md-nav__link">
    Plugins
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#custom" class="md-nav__link">
    Custom
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#precision-sur-lannotation-de-vcf" class="md-nav__link">
    Précision sur l'annotation de VCF
  </a>
  
    <nav class="md-nav" aria-label="Précision sur l'annotation de VCF">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#le-header" class="md-nav__link">
    Le header
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#le-champ-csq" class="md-nav__link">
    Le champ CSQ
  </a>
  
    <nav class="md-nav" aria-label="Le champ CSQ">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#exemple-de-plusieurs-transcrits" class="md-nav__link">
    Exemple de plusieurs transcrits
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#exemple-de-plusieurs-alt" class="md-nav__link">
    Exemple de plusieurs ALT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#reformatage-par-vep" class="md-nav__link">
    Reformatage par VEP
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../databases/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Bases de données
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../filter_strategies/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Stratégies de filtrage
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../acmg/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Classification ACMG
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../technology_watch/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Veille
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
    
    
      
        
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_5" >
        
          
          <label class="md-nav__link" for="__nav_2_5" id="__nav_2_5_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Front-end
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_5">
            <span class="md-nav__icon md-icon"></span>
            Front-end
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_5_1" >
        
          
          <label class="md-nav__link" for="__nav_2_5_1" id="__nav_2_5_1_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Pages
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_2_5_1_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_5_1">
            <span class="md-nav__icon md-icon"></span>
            Pages
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/pages/signin/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Connexion
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/pages/signup/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Création de compte
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/pages/all-pages/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Toutes pages
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/pages/dashboard/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tableau de bord
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/pages/list-analyses/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Liste des analyses
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/pages/list-families/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Liste des familles
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/pages/list-projects/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Liste des projets
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/pages/details-person/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Vue détaillée d'un individu
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/pages/details-family/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Vue détaillée de la famille
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/pages/details-project/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Vue détaillée d'un projet
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/pages/details-analysis/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Accueil d'analyse
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/pages/interpretation-table/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tableau d'interprétation
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
    
    
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_5_2" >
        
          
          <label class="md-nav__link" for="__nav_2_5_2" id="__nav_2_5_2_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Composants
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_2_5_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_5_2">
            <span class="md-nav__icon md-icon"></span>
            Composants
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../front-end/components/table-variants/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tableau de variations
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
    
    
      
        
      
        
      
        
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_6" >
        
          
          <label class="md-nav__link" for="__nav_2_6" id="__nav_2_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Nouveaux arrivants
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_6">
            <span class="md-nav__icon md-icon"></span>
            Nouveaux arrivants
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    
    
      
        
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_6_1" >
        
          
          <label class="md-nav__link" for="__nav_2_6_1" id="__nav_2_6_1_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Fondamentaux
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_2_6_1_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_6_1">
            <span class="md-nav__icon md-icon"></span>
            Fondamentaux
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/objectives/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Fonctionnalités générales
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/organisation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Organisation du projet
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
    
    
      
        
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_6_2" >
        
          
          <label class="md-nav__link" for="__nav_2_6_2" id="__nav_2_6_2_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Fonctionnement de l'appli
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_2_6_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_6_2">
            <span class="md-nav__icon md-icon"></span>
            Fonctionnement de l'appli
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/factories/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Factories
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/analyses/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Lancer une analyse
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
    
    
      
        
      
        
      
        
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_6_3" >
        
          
          <label class="md-nav__link" for="__nav_2_6_3" id="__nav_2_6_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Autour de Diagho
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_2_6_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_6_3">
            <span class="md-nav__icon md-icon"></span>
            Autour de Diagho
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/dna/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    L’ADN
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/interpretation/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Interprétation
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/bioinformatics/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Pipeline bioinformatique
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/variants/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Variations génétiques
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_6_4" >
        
          
          <label class="md-nav__link" for="__nav_2_6_4" id="__nav_2_6_4_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Pour aller plus loin
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_2_6_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_6_4">
            <span class="md-nav__icon md-icon"></span>
            Pour aller plus loin
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/sequencing/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Séquençage
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/somatic/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Génétique somatique
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/rnaseq/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RNA-Seq
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/hugo/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    GCS HUGO
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../newcomers/national_context/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Contexte national
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
    
    
      
        
          
        
      
        
      
        
      
        
      
        
      
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          
          <div class="md-nav__link md-nav__container">
            <a href="../../../user_manual/" class="md-nav__link ">
              
  
  <span class="md-ellipsis">
    User Manual
  </span>
  

            </a>
            
              
              <label class="md-nav__link " for="__nav_3" id="__nav_3_label" tabindex="0">
                <span class="md-nav__icon md-icon"></span>
              </label>
            
          </div>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            User Manual
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../../user_manual/advanced_filters/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Filtres avancés
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../../user_manual/architecture/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Diagho Architecture
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../../user_manual/pathologies/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Pathologies
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../../user_manual/projects/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Les projets
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#ensembl-variant-effect-predictor-vep" class="md-nav__link">
    Ensembl Variant Effect Predictor (VEP)
  </a>
  
    <nav class="md-nav" aria-label="Ensembl Variant Effect Predictor (VEP)">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#output" class="md-nav__link">
    Output
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#annotations-avancees" class="md-nav__link">
    Annotations avancées
  </a>
  
    <nav class="md-nav" aria-label="Annotations avancées">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#plugins" class="md-nav__link">
    Plugins
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#custom" class="md-nav__link">
    Custom
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#precision-sur-lannotation-de-vcf" class="md-nav__link">
    Précision sur l'annotation de VCF
  </a>
  
    <nav class="md-nav" aria-label="Précision sur l'annotation de VCF">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#le-header" class="md-nav__link">
    Le header
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#le-champ-csq" class="md-nav__link">
    Le champ CSQ
  </a>
  
    <nav class="md-nav" aria-label="Le champ CSQ">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#exemple-de-plusieurs-transcrits" class="md-nav__link">
    Exemple de plusieurs transcrits
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#exemple-de-plusieurs-alt" class="md-nav__link">
    Exemple de plusieurs ALT
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#reformatage-par-vep" class="md-nav__link">
    Reformatage par VEP
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<h1 id="les-outils-dannotation">Les outils d'annotation</h1>
<h2 id="ensembl-variant-effect-predictor-vep">Ensembl Variant Effect Predictor (VEP)</h2>
<p><a href="https://www.ensembl.org/info/docs/tools/vep/index.html">VEP</a> est un outil d'annotation de variations développé en Perl sous licence Apache-2.0 au sein du projet Ensembl.<br />
Il permet d'informer quel(s) gène(s) et transcrits sont impactés par une variation et l'impact de cette variation sur ceux-ci mais également d'ajouter des annotations complémentaires inclues dans l'outil de base, récupérées de plugins ou directement d'une source d'informations personnelle sous format VCF, BED, GFF, GTF ou bigWig.</p>
<h3 id="output">Output</h3>
<p>Par défaut son fichier de sortie est un fichier tabulé où toutes les annotations complémentaires sont ajoutées sous la forme clef=valeur dans une colonne <code>Extra</code>, mais il peut annoter directement un vcf ou produire un JSON.</p>
<p>??? abstract "Outputs avec annotation de base"</p>
<div class="highlight"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre><span></span><span class="normal">  1</span>
<span class="normal">  2</span>
<span class="normal">  3</span>
<span class="normal">  4</span>
<span class="normal">  5</span>
<span class="normal">  6</span>
<span class="normal">  7</span>
<span class="normal">  8</span>
<span class="normal">  9</span>
<span class="normal"> 10</span>
<span class="normal"> 11</span>
<span class="normal"> 12</span>
<span class="normal"> 13</span>
<span class="normal"> 14</span>
<span class="normal"> 15</span>
<span class="normal"> 16</span>
<span class="normal"> 17</span>
<span class="normal"> 18</span>
<span class="normal"> 19</span>
<span class="normal"> 20</span>
<span class="normal"> 21</span>
<span class="normal"> 22</span>
<span class="normal"> 23</span>
<span class="normal"> 24</span>
<span class="normal"> 25</span>
<span class="normal"> 26</span>
<span class="normal"> 27</span>
<span class="normal"> 28</span>
<span class="normal"> 29</span>
<span class="normal"> 30</span>
<span class="normal"> 31</span>
<span class="normal"> 32</span>
<span class="normal"> 33</span>
<span class="normal"> 34</span>
<span class="normal"> 35</span>
<span class="normal"> 36</span>
<span class="normal"> 37</span>
<span class="normal"> 38</span>
<span class="normal"> 39</span>
<span class="normal"> 40</span>
<span class="normal"> 41</span>
<span class="normal"> 42</span>
<span class="normal"> 43</span>
<span class="normal"> 44</span>
<span class="normal"> 45</span>
<span class="normal"> 46</span>
<span class="normal"> 47</span>
<span class="normal"> 48</span>
<span class="normal"> 49</span>
<span class="normal"> 50</span>
<span class="normal"> 51</span>
<span class="normal"> 52</span>
<span class="normal"> 53</span>
<span class="normal"> 54</span>
<span class="normal"> 55</span>
<span class="normal"> 56</span>
<span class="normal"> 57</span>
<span class="normal"> 58</span>
<span class="normal"> 59</span>
<span class="normal"> 60</span>
<span class="normal"> 61</span>
<span class="normal"> 62</span>
<span class="normal"> 63</span>
<span class="normal"> 64</span>
<span class="normal"> 65</span>
<span class="normal"> 66</span>
<span class="normal"> 67</span>
<span class="normal"> 68</span>
<span class="normal"> 69</span>
<span class="normal"> 70</span>
<span class="normal"> 71</span>
<span class="normal"> 72</span>
<span class="normal"> 73</span>
<span class="normal"> 74</span>
<span class="normal"> 75</span>
<span class="normal"> 76</span>
<span class="normal"> 77</span>
<span class="normal"> 78</span>
<span class="normal"> 79</span>
<span class="normal"> 80</span>
<span class="normal"> 81</span>
<span class="normal"> 82</span>
<span class="normal"> 83</span>
<span class="normal"> 84</span>
<span class="normal"> 85</span>
<span class="normal"> 86</span>
<span class="normal"> 87</span>
<span class="normal"> 88</span>
<span class="normal"> 89</span>
<span class="normal"> 90</span>
<span class="normal"> 91</span>
<span class="normal"> 92</span>
<span class="normal"> 93</span>
<span class="normal"> 94</span>
<span class="normal"> 95</span>
<span class="normal"> 96</span>
<span class="normal"> 97</span>
<span class="normal"> 98</span>
<span class="normal"> 99</span>
<span class="normal">100</span>
<span class="normal">101</span>
<span class="normal">102</span>
<span class="normal">103</span>
<span class="normal">104</span>
<span class="normal">105</span>
<span class="normal">106</span>
<span class="normal">107</span>
<span class="normal">108</span>
<span class="normal">109</span>
<span class="normal">110</span>
<span class="normal">111</span>
<span class="normal">112</span>
<span class="normal">113</span>
<span class="normal">114</span>
<span class="normal">115</span>
<span class="normal">116</span>
<span class="normal">117</span>
<span class="normal">118</span>
<span class="normal">119</span>
<span class="normal">120</span>
<span class="normal">121</span>
<span class="normal">122</span>
<span class="normal">123</span>
<span class="normal">124</span>
<span class="normal">125</span>
<span class="normal">126</span>
<span class="normal">127</span>
<span class="normal">128</span>
<span class="normal">129</span>
<span class="normal">130</span>
<span class="normal">131</span>
<span class="normal">132</span>
<span class="normal">133</span>
<span class="normal">134</span>
<span class="normal">135</span>
<span class="normal">136</span>
<span class="normal">137</span>
<span class="normal">138</span>
<span class="normal">139</span>
<span class="normal">140</span>
<span class="normal">141</span>
<span class="normal">142</span>
<span class="normal">143</span>
<span class="normal">144</span>
<span class="normal">145</span>
<span class="normal">146</span>
<span class="normal">147</span>
<span class="normal">148</span>
<span class="normal">149</span>
<span class="normal">150</span>
<span class="normal">151</span>
<span class="normal">152</span>
<span class="normal">153</span>
<span class="normal">154</span>
<span class="normal">155</span>
<span class="normal">156</span>
<span class="normal">157</span>
<span class="normal">158</span>
<span class="normal">159</span>
<span class="normal">160</span>
<span class="normal">161</span>
<span class="normal">162</span>
<span class="normal">163</span>
<span class="normal">164</span>
<span class="normal">165</span>
<span class="normal">166</span>
<span class="normal">167</span>
<span class="normal">168</span>
<span class="normal">169</span>
<span class="normal">170</span>
<span class="normal">171</span>
<span class="normal">172</span>
<span class="normal">173</span>
<span class="normal">174</span>
<span class="normal">175</span>
<span class="normal">176</span>
<span class="normal">177</span>
<span class="normal">178</span>
<span class="normal">179</span>
<span class="normal">180</span>
<span class="normal">181</span>
<span class="normal">182</span>
<span class="normal">183</span>
<span class="normal">184</span>
<span class="normal">185</span>
<span class="normal">186</span>
<span class="normal">187</span>
<span class="normal">188</span>
<span class="normal">189</span>
<span class="normal">190</span>
<span class="normal">191</span>
<span class="normal">192</span>
<span class="normal">193</span>
<span class="normal">194</span>
<span class="normal">195</span>
<span class="normal">196</span>
<span class="normal">197</span>
<span class="normal">198</span></pre></div></td><td class="code"><div><pre><span></span><code>=== &quot;Standard&quot;

    **Commande**
    ```
    ./vep --input_file /opt/vep/.vep/inputs/sample.vcf --output_file /opt/vep/.vep/outputs/std_output --cache --offline --format vcf
    ```

    **Output**
    ```
    ## ENSEMBL VARIANT EFFECT PREDICTOR v109.3
    ## Output produced at 2023-04-14 11:45:52
    ## Using cache in /opt/vep/.vep/homo_sapiens/109_GRCh37
    ## Using API version 109, DB version ?
    ## ensembl version 109.10baaec
    ## ensembl-variation version 109.18a12b6
    ## ensembl-funcgen version 109.cba2db8
    ## ensembl-io version 109.4946a86
    ## genebuild version 2011-04
    ## 1000genomes version phase3
    ## HGMD-PUBLIC version 20204
    ## dbSNP version 154
    ## COSMIC version 92
    ## ClinVar version 202012
    ## gencode version GENCODE 19
    ## gnomADe version r2.1
    ## sift version sift5.2.2
    ## assembly version GRCh37.p13
    ## regbuild version 1.0
    ## polyphen version 2.2.2
    ## Column descriptions:
    ## Uploaded_variation : Identifier of uploaded variant
    ## Location : Location of variant in standard coordinate format (chr:start or chr:start-end)
    ## Allele : The variant allele used to calculate the consequence
    ## Gene : Stable ID of affected gene
    ## Feature : Stable ID of feature
    ## Feature_type : Type of feature - Transcript, RegulatoryFeature or MotifFeature
    ## Consequence : Consequence type
    ## cDNA_position : Relative position of base pair in cDNA sequence
    ## CDS_position : Relative position of base pair in coding sequence
    ## Protein_position : Relative position of amino acid in protein
    ## Amino_acids : Reference and variant amino acids
    ## Codons : Reference and variant codon sequence
    ## Existing_variation : Identifier(s) of co-located known variants
    ## Extra column keys:
    ## IMPACT : Subjective impact classification of consequence type
    ## DISTANCE : Shortest distance from variant to transcript
    ## STRAND : Strand of the feature (1/-1)
    ## FLAGS : Transcript quality flags
    ## VEP command-line: vep --cache --database 0 --dir_plugins /plugins --format vcf --input_file [PATH]/sample.vcf --offline --output_file [PATH]/std_output
    #Uploaded_variation Location    Allele  Gene    Feature Feature_type    Consequence cDNA_position   CDS_position    Protein_position    Amino_acids Codons  Existing_variation  Extra
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000379370 Transcript  synonymous_variant  4211    4161    1387    T   acT/acC -   IMPACT=LOW;STRAND=1
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000419249 Transcript  upstream_gene_variant   -   -   -   -   -   -   IMPACT=MODIFIER;DISTANCE=1634;STRAND=1;FLAGS=cds_start_NF,cds_end_NF
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000461111 Transcript  upstream_gene_variant   -   -   -   -   -   -   IMPACT=MODIFIER;DISTANCE=3064;STRAND=1
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000466223 Transcript  downstream_gene_variant -   -   -   -   -   -   IMPACT=MODIFIER;DISTANCE=1057;STRAND=1
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000478677 Transcript  downstream_gene_variant -   -   -   -   -   -   IMPACT=MODIFIER;DISTANCE=1084;STRAND=1
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000479707 Transcript  downstream_gene_variant -   -   -   -   -   -   IMPACT=MODIFIER;DISTANCE=2573;STRAND=1
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000492947 Transcript  non_coding_transcript_exon_variant  394 -   -   -   -   -   IMPACT=MODIFIER;STRAND=1
    chr1_985266_C/T chr1:985266 T   ENSG00000188157 ENST00000379370 Transcript  splice_polypyrimidine_tract_variant,intron_variant  -   -   -   -   -   -   IMPACT=LOW;STRAND=1
    chr1_985266_C/T chr1:985266 T   ENSG00000188157 ENST00000419249 Transcript  upstream_gene_variant   -   -   -   -   -   -   IMPACT=MODIFIER;DISTANCE=670;STRAND=1;FLAGS=cds_start_NF,cds_end_NF
    ```


=== &quot;VCF&quot;

    **Commande**
    ```
    ./vep --input_file /opt/vep/.vep/inputs/sample.vcf --output_file /opt/vep/.vep/outputs/std_output.vcf --cache --offline --everything --format vcf --vcf
    ```

    **Note**

    - Si une annotation contient une `,` elle sera remplacée par une `&amp;` dans le VCF produit.
    - Pour cet exemple, toutes les annotations disponibles de base dans VEP ont été appliquées.

    **Output**
    ```
    ##fileformat=VCFv4.2
    ##FILTER=&lt;ID=LowCoverage,Description=&quot;DP &lt; 5&quot;&gt;
    ##FILTER=&lt;ID=SNP_filter,Description=&quot;DP &lt; 10 || QD &lt; 2.0 || FS &gt; 60.0 || MQ &lt; 40.0 || FBtypeScore &gt; 13.0 || MappingQualityRankSum &lt; -12.5 || ReadPosRankSum &lt; -8.0&quot;&gt;
    ##FORMAT=&lt;ID=AD,Number=R,Type=Integer,Description=&quot;Number of observation for each allele&quot;&gt;
    ##FORMAT=&lt;ID=AO,Number=A,Type=Integer,Description=&quot;Alternate allele observation count&quot;&gt;
    ##FORMAT=&lt;ID=DP,Number=1,Type=Integer,Description=&quot;Read Depth&quot;&gt;
    ##FORMAT=&lt;ID=GT,Number=1,Type=String,Description=&quot;Genotype&quot;&gt;
    ##FORMAT=&lt;ID=PL,Number=G,Type=Integer,Description=&quot;Normalized, Phred-scaled likelihoods for genotypes as defined in the VCF specification&quot;&gt;
    ##FORMAT=&lt;ID=QA,Number=A,Type=Integer,Description=&quot;Sum of quality of the alternate observations&quot;&gt;
    ##FORMAT=&lt;ID=QR,Number=1,Type=Integer,Description=&quot;Sum of quality of the reference observations&quot;&gt;
    ##FORMAT=&lt;ID=RO,Number=1,Type=Integer,Description=&quot;Reference allele observation count&quot;&gt;
    ##INFO=&lt;ID=AB,Number=A,Type=Float,Description=&quot;Allele balance at heterozygous sites: a number between 0 and 1 representing the ratio of reads showing the reference allele to all reads, considering only reads from individuals called as heterozygous&quot;&gt;
    ##INFO=&lt;ID=ABP,Number=A,Type=Float,Description=&quot;Allele balance probability at heterozygous sites: Phred-scaled upper-bounds estimate of the probability of observing the deviation between ABR and ABA given E(ABR/ABA) ~ 0.5, derived using Hoeffding&#39;s inequality&quot;&gt;
    ##INFO=&lt;ID=AC,Number=A,Type=Integer,Description=&quot;Allele count in genotypes, for each ALT allele, in the same order as listed&quot;&gt;
    ##INFO=&lt;ID=AF,Number=A,Type=Float,Description=&quot;Allele Frequency, for each ALT allele, in the same order as listed&quot;&gt;
    ##INFO=&lt;ID=AN,Number=1,Type=Integer,Description=&quot;Total number of alleles in called genotypes&quot;&gt;
    ##INFO=&lt;ID=AO,Number=A,Type=Integer,Description=&quot;Count of full observations of this alternate haplotype.&quot;&gt;
    ##INFO=&lt;ID=CIGAR,Number=A,Type=String,Description=&quot;The extended CIGAR representation of each alternate allele, with the exception that &#39;=&#39; is replaced by &#39;M&#39; to ease VCF parsing.  Note that INDEL alleles do not have the first matched base (which is provided by default, per the spec) referred to by the CIGAR.&quot;&gt;
    ##INFO=&lt;ID=DP,Number=1,Type=Integer,Description=&quot;Approximate read depth; some reads may have been filtered&quot;&gt;
    ##VEP=&quot;v109&quot; time=&quot;2023-04-14 12:14:01&quot; cache=&quot;/opt/vep/.vep/homo_sapiens/109_GRCh37&quot; ensembl-io=109.4946a86 ensembl=109.10baaec ensembl-funcgen=109.cba2db8 ensembl-variation=109.18a12b6 1000genomes=&quot;phase3&quot; COSMIC=&quot;92&quot; ClinVar=&quot;202012&quot; HGMD-PUBLIC=&quot;20204&quot; assembly=&quot;GRCh37.p13&quot; dbSNP=&quot;154&quot; gencode=&quot;GENCODE 19&quot; genebuild=&quot;2011-04&quot; gnomADe=&quot;r2.1&quot; polyphen=&quot;2.2.2&quot; regbuild=&quot;1.0&quot; sift=&quot;sift5.2.2&quot;
    ##INFO=&lt;ID=CSQ,Number=.,Type=String,Description=&quot;Consequence annotations from Ensembl VEP. Format: Allele|Consequence|IMPACT|SYMBOL|Gene|Feature_type|Feature|BIOTYPE|EXON|INTRON|HGVSc|HGVSp|cDNA_position|CDS_position|Protein_position|Amino_acids|Codons|Existing_variation|DISTANCE|STRAND|FLAGS|VARIANT_CLASS|SYMBOL_SOURCE|HGNC_ID|CANONICAL|MANE_SELECT|MANE_PLUS_CLINICAL|TSL|APPRIS|CCDS|ENSP|SWISSPROT|TREMBL|UNIPARC|UNIPROT_ISOFORM|GENE_PHENO|SIFT|PolyPhen|DOMAINS|miRNA|HGVS_OFFSET|AF|AFR_AF|AMR_AF|EAS_AF|EUR_AF|SAS_AF|gnomADe_AF|gnomADe_AFR_AF|gnomADe_AMR_AF|gnomADe_ASJ_AF|gnomADe_EAS_AF|gnomADe_FIN_AF|gnomADe_NFE_AF|gnomADe_OTH_AF|gnomADe_SAS_AF|gnomADg_AF|gnomADg_AFR_AF|gnomADg_AMI_AF|gnomADg_AMR_AF|gnomADg_ASJ_AF|gnomADg_EAS_AF|gnomADg_FIN_AF|gnomADg_MID_AF|gnomADg_NFE_AF|gnomADg_OTH_AF|gnomADg_SAS_AF|MAX_AF|MAX_AF_POPS|CLIN_SIG|SOMATIC|PHENO|PUBMED|MOTIF_NAME|MOTIF_POS|HIGH_INF_POS|MOTIF_SCORE_CHANGE|TRANSCRIPTION_FACTORS&quot;&gt;
    ##VEP-command-line=&#39;vep --cache --database 0 --dir_plugins /plugins --everything --format vcf --input_file [PATH]/sample.vcf --offline --output_file [PATH]/std_output.vcf --vcf&#39;
    #CHROM  POS ID  REF ALT QUAL    FILTER  INFO    FORMAT  22-23121-A-02-00    22-23123-A-02-00    22-24543-A-01-00
    chr1    984302  .   T   C   3877.41 PASS    AB=0.464088;ABP=5.03781;AC=4;AF=0.667;AN=6;AO=151;CIGAR=1X;DP=249;CSQ=C|synonymous_variant|LOW|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|24/36||ENST00000379370.2:c.4161T&gt;C|ENSP00000368678.2:p.Thr1387%3D|4211|4161|1387|T|acT/acC|rs9442391||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1|||Gene3D:2.60.120.200&amp;PROSITE_profiles:PS50025&amp;PANTHER:PTHR10574&amp;PANTHER:PTHR10574:SF227&amp;Superfamily:SSF49899|||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&amp;25741868|||||,C|upstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs9442391|1634|1|cds_start_NF&amp;cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&amp;25741868|||||,C|upstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron||||||||||rs9442391|3064|1||SNV|HGNC|329||||||||||||1||||||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&amp;25741868|||||,C|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000466223|retained_intron||||||||||rs9442391|1057|1||SNV|HGNC|329||||||||||||1||||||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&amp;25741868|||||,C|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000478677|retained_intron||||||||||rs9442391|1084|1||SNV|HGNC|329||||||||||||1||||||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&amp;25741868|||||,C|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000479707|retained_intron||||||||||rs9442391|2573|1||SNV|HGNC|329||||||||||||1||||||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&amp;25741868|||||,C|non_coding_transcript_exon_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000492947|retained_intron|1/2||ENST00000492947.1:n.394T&gt;C||394|||||rs9442391||1||SNV|HGNC|329||||||||||||1||||||0.5457|0.2405|0.6614|0.8294|0.5636|0.5654|0.5883|0.2729|0.705|0.5642|0.8196|0.5423|0.5652|0.5716|0.5594||||||||||||0.8294|EAS|benign||1|18414213&amp;25741868|||||   GT:AD:AO:DP:PL:QA:QR:RO 1/1:1,67:67:68:1979,171,0:2234:34:1 0/1:52,35:35:87:812,0,1296:1190:1728:52 0/1:45,49:49:94:1219,0,1056:1666:1484:45
    chr1    985266  .   C   T   2801.75 PASS    AB=0.491667;ABP=3.08268;AC=4;AF=0.667;AN=6;AO=110;CIGAR=1X;DP=171;CSQ=T|splice_polypyrimidine_tract_variant&amp;intron_variant|LOW|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding||26/35|ENST00000379370.2:c.4745-17C&gt;T|||||||rs2275813||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||,T|upstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs2275813|670|1|cds_start_NF&amp;cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||,T|upstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron||||||||||rs2275813|2100|1||SNV|HGNC|329||||||||||||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000466223|retained_intron||||||||||rs2275813|2021|1||SNV|HGNC|329||||||||||||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000478677|retained_intron||||||||||rs2275813|2048|1||SNV|HGNC|329||||||||||||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000479707|retained_intron||||||||||rs2275813|3537|1||SNV|HGNC|329||||||||||||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000492947|retained_intron||||||||||rs2275813|492|1||SNV|HGNC|329||||||||||||1||||||0.4952|0.1021|0.6398|0.8155|0.5457|0.5419|0.5575|0.148|0.7111|0.5442|0.8103|0.5294|0.5353|0.542|0.5454||||||||||||0.8155|EAS|benign||1|25741868|||||    GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,51:51:51:1549,154,0:1718:0:0  0/1:33,27:27:60:649,0,825:918:1114:33   0/1:28,32:32:60:782,0,679:1066:952:28
    chr1    986732  .   G   A   922.52  PASS    AB=0.415094;ABP=9.64763;AC=1;AF=0.167;AN=6;AO=44;CIGAR=1X;DP=356;CSQ=A|missense_variant|MODERATE|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|31/36||ENST00000379370.2:c.5353G&gt;A|ENSP00000368678.2:p.Asp1785Asn|5403|5353|1785|D/N|Gac/Aac|rs144245019&amp;COSV99062175||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1|tolerated(0.27)|benign(0)|Gene3D:2.60.120.200&amp;Pfam:PF00054&amp;PROSITE_profiles:PS50025&amp;PANTHER:PTHR10574&amp;PANTHER:PTHR10574:SF227&amp;SMART:SM00282&amp;Superfamily:SSF49899|||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&amp;uncertain_significance|0&amp;1|1&amp;1|18414213&amp;25741868|||||,A|upstream_gene_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic||||||||||rs144245019&amp;COSV99062175|3681|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&amp;uncertain_significance|0&amp;1|1&amp;1|18414213&amp;25741868|||||,A|missense_variant|MODERATE|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding|4/9||ENST00000419249.1:c.262G&gt;A|ENSP00000400771.1:p.Asp88Asn|260|262|88|D/N|Gac/Aac|rs144245019&amp;COSV99062175||1|cds_start_NF&amp;cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1|tolerated(0.48)|benign(0.006)|Gene3D:2.60.120.200&amp;Pfam:PF00054&amp;PROSITE_profiles:PS50025&amp;PANTHER:PTHR10574&amp;PANTHER:PTHR10574:SF227&amp;Superfamily:SSF49899|||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&amp;uncertain_significance|0&amp;1|1&amp;1|18414213&amp;25741868|||||,A|upstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron||||||||||rs144245019&amp;COSV99062175|634|1||SNV|HGNC|329||||||||||||1||||||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&amp;uncertain_significance|0&amp;1|1&amp;1|18414213&amp;25741868|||||,A|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000466223|retained_intron||||||||||rs144245019&amp;COSV99062175|3487|1||SNV|HGNC|329||||||||||||1||||||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&amp;uncertain_significance|0&amp;1|1&amp;1|18414213&amp;25741868|||||,A|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000478677|retained_intron||||||||||rs144245019&amp;COSV99062175|3514|1||SNV|HGNC|329||||||||||||1||||||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&amp;uncertain_significance|0&amp;1|1&amp;1|18414213&amp;25741868|||||,A|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000492947|retained_intron||||||||||rs144245019&amp;COSV99062175|1958|1||SNV|HGNC|329||||||||||||1||||||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&amp;uncertain_significance|0&amp;1|1&amp;1|18414213&amp;25741868|||||,A|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00001771209|CTCF_binding_site||||||||||rs144245019&amp;COSV99062175||||SNV||||||||||||||||||||0.0018|0.0008|0.0058|0|0.002|0.002|0.006331|0.001643|0.004431|0.007636|0|0.003146|0.01139|0.006242|0.0009532||||||||||||0.01139|gnomADe_NFE|benign&amp;uncertain_significance|0&amp;1|1&amp;1|18414213&amp;25741868|||||  GT:AD:AO:DP:PL:QA:QR:RO 0/0:129,0:0:129:0,388,3908:0:4340:129   0/1:62,44:44:106:1030,0,1581:1496:2108:62   0/0:121,0:0:121:0,364,3683:0:4090:121
    chr1    987200  .   C   T   6412.21 PASS    AB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=219;CIGAR=1X;DP=219;CSQ=T|splice_donor_5th_base_variant&amp;intron_variant|LOW|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding||33/35|ENST00000379370.2:c.5651+5C&gt;T|||||||rs9803031||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&amp;25741868|||||,T|upstream_gene_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic||||||||||rs9803031|3213|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&amp;25741868|||||,T|splice_donor_5th_base_variant&amp;intron_variant|LOW|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||6/8|ENST00000419249.1:c.558+5C&gt;T|||||||rs9803031||1|cds_start_NF&amp;cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&amp;25741868|||||,T|upstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron||||||||||rs9803031|166|1||SNV|HGNC|329||||||||||||1||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&amp;25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000466223|retained_intron||||||||||rs9803031|3955|1||SNV|HGNC|329||||||||||||1||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&amp;25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000478677|retained_intron||||||||||rs9803031|3982|1||SNV|HGNC|329||||||||||||1||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&amp;25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000492947|retained_intron||||||||||rs9803031|2426|1||SNV|HGNC|329||||||||||||1||||||0.7887|0.4017|0.8876|0.9931|0.9145|0.9018|0.8871|0.4387|0.932|0.8923|0.9957|0.9505|0.9082|0.8928|0.8828||||||||||||0.9957|gnomADe_EAS|benign||1|18414213&amp;25741868|||||    GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,63:63:63:1911,190,0:2120:0:0  1/1:0,66:66:66:1963,199,0:2178:0:0  1/1:0,90:90:90:2737,271,0:3038:0:0
    chr1    990280  .   C   T   4547.65 PASS    AB=0.446602;ABP=8.11221;AC=4;AF=0.667;AN=6;AO=175;CIGAR=1X;DP=289;CSQ=T|synonymous_variant|LOW|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|36/36||ENST00000379370.2:c.6057C&gt;T|ENSP00000368678.2:p.Asp2019%3D|6107|6057|2019|D|gaC/gaT|rs4275402&amp;COSV65070964||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1|||Gene3D:2.60.120.200&amp;Pfam:PF00054&amp;PROSITE_profiles:PS50025&amp;PANTHER:PTHR10574&amp;PANTHER:PTHR10574:SF227&amp;SMART:SM00282&amp;Superfamily:SSF49899|||0.5853|0.2617|0.7262|0.8214|0.6779|0.5838|0.6764|0.3114|0.7722|0.6575|0.8138|0.641|0.698|0.6758|0.6133||||||||||||0.8214|EAS|benign|0&amp;1|1&amp;1|18414213&amp;25741868|||||,T|upstream_gene_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic||||||||||rs4275402&amp;COSV65070964|133|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.5853|0.2617|0.7262|0.8214|0.6779|0.5838|0.6764|0.3114|0.7722|0.6575|0.8138|0.641|0.698|0.6758|0.6133||||||||||||0.8214|EAS|benign|0&amp;1|1&amp;1|18414213&amp;25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs4275402&amp;COSV65070964|950|1|cds_start_NF&amp;cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.5853|0.2617|0.7262|0.8214|0.6779|0.5838|0.6764|0.3114|0.7722|0.6575|0.8138|0.641|0.698|0.6758|0.6133||||||||||||0.8214|EAS|benign|0&amp;1|1&amp;1|18414213&amp;25741868|||||,T|upstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000427998|processed_transcript||||||||||rs4275402&amp;COSV65070964|4834|1||SNV|Clone_based_vega_gene|||||||||||||||||||0.5853|0.2617|0.7262|0.8214|0.6779|0.5838|0.6764|0.3114|0.7722|0.6575|0.8138|0.641|0.698|0.6758|0.6133||||||||||||0.8214|EAS|benign|0&amp;1|1&amp;1|18414213&amp;25741868|||||,T|non_coding_transcript_exon_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron|3/3||ENST00000461111.1:n.2173C&gt;T||2173|||||rs4275402&amp;COSV65070964||1||SNV|HGNC|329||||||||||||1||||||0.5853|0.2617|0.7262|0.8214|0.6779|0.5838|0.6764|0.3114|0.7722|0.6575|0.8138|0.641|0.698|0.6758|0.6133||||||||||||0.8214|EAS|benign|0&amp;1|1&amp;1|18414213&amp;25741868|||||,T|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00001771211|CTCF_binding_site||||||||||rs4275402&amp;COSV65070964||||SNV||||||||||||||||||||0.5853|0.2617|0.7262|0.8214|0.6779|0.5838|0.6764|0.3114|0.7722|0.6575|0.8138|0.641|0.698|0.6758|0.6133||||||||||||0.8214|EAS|benign|0&amp;1|1&amp;1|18414213&amp;25741868||||| GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,83:83:83:2496,250,0:2770:0:0  0/1:56,39:39:95:910,0,1371:1326:1838:56 0/1:58,53:53:111:1283,0,1436:1794:1964:58
    chr1    990380  .   C   T   570.29  PASS    AB=0.55102;ABP=4.11819;AC=2;AF=0.333;AN=6;AO=27;CIGAR=1X;DP=63;CSQ=T|3_prime_UTR_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|36/36||ENST00000379370.2:c.*19C&gt;T||6207|||||rs3121561&amp;COSV65069616||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1||||||0.3434|0.5461|0.2594|0.1726|0.3091|0.3405|0.2913|0.5233|0.2137|0.3111|0.1794|0.3513|0.292|0.2986|0.3087||||||||||||0.5461|AFR|benign|0&amp;1|1&amp;1|25741868|||||,T|upstream_gene_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic||||||||||rs3121561&amp;COSV65069616|33|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.3434|0.5461|0.2594|0.1726|0.3091|0.3405|0.2913|0.5233|0.2137|0.3111|0.1794|0.3513|0.292|0.2986|0.3087||||||||||||0.5461|AFR|benign|0&amp;1|1&amp;1|25741868|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs3121561&amp;COSV65069616|1050|1|cds_start_NF&amp;cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.3434|0.5461|0.2594|0.1726|0.3091|0.3405|0.2913|0.5233|0.2137|0.3111|0.1794|0.3513|0.292|0.2986|0.3087||||||||||||0.5461|AFR|benign|0&amp;1|1&amp;1|25741868|||||,T|upstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000427998|processed_transcript||||||||||rs3121561&amp;COSV65069616|4734|1||SNV|Clone_based_vega_gene|||||||||||||||||||0.3434|0.5461|0.2594|0.1726|0.3091|0.3405|0.2913|0.5233|0.2137|0.3111|0.1794|0.3513|0.292|0.2986|0.3087||||||||||||0.5461|AFR|benign|0&amp;1|1&amp;1|25741868|||||,T|non_coding_transcript_exon_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron|3/3||ENST00000461111.1:n.2273C&gt;T||2273|||||rs3121561&amp;COSV65069616||1||SNV|HGNC|329||||||||||||1||||||0.3434|0.5461|0.2594|0.1726|0.3091|0.3405|0.2913|0.5233|0.2137|0.3111|0.1794|0.3513|0.292|0.2986|0.3087||||||||||||0.5461|AFR|benign|0&amp;1|1&amp;1|25741868|||||,T|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00001771211|CTCF_binding_site||||||||||rs3121561&amp;COSV65069616||||SNV||||||||||||||||||||0.3434|0.5461|0.2594|0.1726|0.3091|0.3405|0.2913|0.5233|0.2137|0.3111|0.1794|0.3513|0.292|0.2986|0.3087||||||||||||0.5461|AFR|benign|0&amp;1|1&amp;1|25741868|||||   GT:AD:AO:DP:PL:QA:QR:RO 0/0:14,0:0:14:0,42,432:0:476:14 0/1:8,14:14:22:365,0,182:476:272:8  0/1:14,13:13:27:320,0,343:442:468:14
    chr1    990417  .   T   C   328.36  PASS    AB=0.580645;ABP=4.76149;AC=2;AF=0.333;AN=6;AO=18;CIGAR=1X;DP=41;CSQ=C|3_prime_UTR_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|36/36||ENST00000379370.2:c.*56T&gt;C||6244|||||rs2465136&amp;COSV65069776||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1||||||0.3840|0.6475|0.2723|0.1786|0.3211|0.3834|||||||||||||||||||||0.6475|AFR||0&amp;1|0&amp;1||||||,C|non_coding_transcript_exon_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic|1/2||ENST00000418300.1:n.5T&gt;C||5|||||rs2465136&amp;COSV65069776||1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.3840|0.6475|0.2723|0.1786|0.3211|0.3834|||||||||||||||||||||0.6475|AFR||0&amp;1|0&amp;1||||||,C|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs2465136&amp;COSV65069776|1087|1|cds_start_NF&amp;cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.3840|0.6475|0.2723|0.1786|0.3211|0.3834|||||||||||||||||||||0.6475|AFR||0&amp;1|0&amp;1||||||,C|upstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000427998|processed_transcript||||||||||rs2465136&amp;COSV65069776|4697|1||SNV|Clone_based_vega_gene|||||||||||||||||||0.3840|0.6475|0.2723|0.1786|0.3211|0.3834|||||||||||||||||||||0.6475|AFR||0&amp;1|0&amp;1||||||,C|non_coding_transcript_exon_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron|3/3||ENST00000461111.1:n.2310T&gt;C||2310|||||rs2465136&amp;COSV65069776||1||SNV|HGNC|329||||||||||||1||||||0.3840|0.6475|0.2723|0.1786|0.3211|0.3834|||||||||||||||||||||0.6475|AFR||0&amp;1|0&amp;1||||||  GT:AD:AO:DP:PL:QA:QR:RO 0/0:10,0:0:10:0,30,309:0:340:10 0/1:5,9:9:14:237,0,114:306:170:5    0/1:8,9:9:17:220,0,197:298:272:8
    chr1    990517  .   C   T   182.67  SNP_filter  AB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=8;CIGAR=1X;DP=8;CSQ=T|3_prime_UTR_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|36/36||ENST00000379370.2:c.*156C&gt;T||6344|||||rs2710872||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1||||||0.9179|0.8419|0.928|0.999|0.8877|0.9611|||||||||||||||||||||0.999|EAS|||1|32314116|||||,T|non_coding_transcript_exon_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic|1/2||ENST00000418300.1:n.105C&gt;T||105|||||rs2710872||1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.9179|0.8419|0.928|0.999|0.8877|0.9611|||||||||||||||||||||0.999|EAS|||1|32314116|||||,T|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs2710872|1187|1|cds_start_NF&amp;cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.9179|0.8419|0.928|0.999|0.8877|0.9611|||||||||||||||||||||0.999|EAS|||1|32314116|||||,T|upstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000427998|processed_transcript||||||||||rs2710872|4597|1||SNV|Clone_based_vega_gene|||||||||||||||||||0.9179|0.8419|0.928|0.999|0.8877|0.9611|||||||||||||||||||||0.999|EAS|||1|32314116|||||,T|non_coding_transcript_exon_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron|3/3||ENST00000461111.1:n.2410C&gt;T||2410|||||rs2710872||1||SNV|HGNC|329||||||||||||1||||||0.9179|0.8419|0.928|0.999|0.8877|0.9611|||||||||||||||||||||0.999|EAS|||1|32314116|||||  GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,1:1:1:34,3,0:34:0:0   1/1:0,4:4:4:126,12,0:136:0:0    1/1:0,3:3:3:95,9,0:102:0:0
    chr1    990984  .   G   A   82.39   LowCoverage;SNP_filter  AB=0;ABP=0;AC=2;AF=1.00;AN=2;AO=3;CIGAR=1X;DP=3;CSQ=A|3_prime_UTR_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000379370|protein_coding|36/36||ENST00000379370.2:c.*623G&gt;A||6811|||||rs8014||1||SNV|HGNC|329|YES|||||CCDS30551.1|ENSP00000368678|AGRIN_HUMAN|Q5XG79_HUMAN|UPI00001D7C8B||1||||||0.9223|0.8555|0.9323|0.999|0.8887|0.9611|||||||||||||||||||||0.999|EAS|||||||||,A|intron_variant&amp;non_coding_transcript_variant|MODIFIER|RP11-54O7.14|ENSG00000242590|Transcript|ENST00000418300|sense_intronic||1/1|ENST00000418300.1:n.184-294G&gt;A|||||||rs8014||1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.9223|0.8555|0.9323|0.999|0.8887|0.9611|||||||||||||||||||||0.999|EAS|||||||||,A|downstream_gene_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000419249|protein_coding||||||||||rs8014|1654|1|cds_start_NF&amp;cds_end_NF|SNV|HGNC|329|||||||ENSP00000400771|||UPI000059CF46||1||||||0.9223|0.8555|0.9323|0.999|0.8887|0.9611|||||||||||||||||||||0.999|EAS|||||||||,A|upstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000427998|processed_transcript||||||||||rs8014|4130|1||SNV|Clone_based_vega_gene|||||||||||||||||||0.9223|0.8555|0.9323|0.999|0.8887|0.9611|||||||||||||||||||||0.999|EAS|||||||||,A|non_coding_transcript_exon_variant|MODIFIER|AGRN|ENSG00000188157|Transcript|ENST00000461111|retained_intron|3/3||ENST00000461111.1:n.2877G&gt;A||2877|||||rs8014||1||SNV|HGNC|329||||||||||||1||||||0.9223|0.8555|0.9323|0.999|0.8887|0.9611|||||||||||||||||||||0.999|EAS|||||||||  GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,3:3:3:95,9,0:102:0:0  .   .
    chr1    1007203 .   A   G   3110.06 PASS    AB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=105;CIGAR=1X;DP=105;CSQ=G|downstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000412397|transcribed_unprocessed_pseudogene||||||||||rs4633229|2468|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.8413|0.8797|0.8112|0.9196|0.7883|0.7843|0.816|0.8594|0.8533|0.8069|0.9078|0.8538|0.7842|0.8041|0.7851||||||||||||0.9196|EAS|||||||||,G|synonymous_variant|LOW|RNF223|ENSG00000237330|Transcript|ENST00000453464|protein_coding|2/2||ENST00000453464.2:c.744T&gt;C|ENSP00000410533.1:p.Asp248%3D|1045|744|248|D|gaT/gaC|rs4633229||-1||SNV|HGNC|40020|YES|||||CCDS53257.1|ENSP00000410533|RN223_HUMAN||UPI0001A5E6EF|||||PANTHER:PTHR22791&amp;PANTHER:PTHR22791:SF4|||0.8413|0.8797|0.8112|0.9196|0.7883|0.7843|0.816|0.8594|0.8533|0.8069|0.9078|0.8538|0.7842|0.8041|0.7851||||||||||||0.9196|EAS|||||||||    GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,26:26:26:752,78,0:832:0:0 1/1:0,46:46:46:1411,138,0:1564:0:0  1/1:0,33:33:33:1013,99,0:1122:0:0
    chr1    1007222 .   G   T   1780.81 PASS    AB=0.685185;ABP=35.1803;AC=2;AF=0.333;AN=6;AO=74;CIGAR=1X;DP=143;CSQ=T|downstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000412397|transcribed_unprocessed_pseudogene||||||||||rs71628928&amp;COSV69251322|2487|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.1174|0.0257|0.0951|0.1091|0.163|0.2188|0.1412|0.02294|0.07825|0.1934|0.07649|0.2412|0.148|0.1606|0.2119||||||||||||0.2412|gnomADe_FIN||0&amp;1|0&amp;1||||||,T|missense_variant|MODERATE|RNF223|ENSG00000237330|Transcript|ENST00000453464|protein_coding|2/2||ENST00000453464.2:c.725C&gt;A|ENSP00000410533.1:p.Pro242His|1026|725|242|P/H|cCc/cAc|rs71628928&amp;COSV69251322||-1||SNV|HGNC|40020|YES|||||CCDS53257.1|ENSP00000410533|RN223_HUMAN||UPI0001A5E6EF|||tolerated(0.22)|possibly_damaging(0.897)|Low_complexity_(Seg):seg&amp;PANTHER:PTHR22791&amp;PANTHER:PTHR22791:SF4|||0.1174|0.0257|0.0951|0.1091|0.163|0.2188|0.1412|0.02294|0.07825|0.1934|0.07649|0.2412|0.148|0.1606|0.2119||||||||||||0.2412|gnomADe_FIN||0&amp;1|0&amp;1||||||  GT:AD:AO:DP:PL:QA:QR:RO 0/0:35,0:0:35:0,105,1034:0:1146:35  0/1:16,40:40:56:1044,0,277:1344:492:16  0/1:18,34:34:52:887,0,397:1156:612:18
    chr1    1007432 .   G   A   3295.23 PASS    AB=0.44375;ABP=7.40753;AC=4;AF=0.667;AN=6;AO=128;CIGAR=1X;DP=217;CSQ=A|downstream_gene_variant|MODIFIER|RP11-465B22.3|ENSG00000217801|Transcript|ENST00000412397|transcribed_unprocessed_pseudogene||||||||||rs4333796&amp;COSV69251112|2697|1||SNV|Clone_based_vega_gene||YES|||||||||||||||||0.5321|0.2307|0.6643|0.7917|0.5616|0.5481|0.6458|0.3916|0.7639|0.6074|0.8456|0.5287|0.6098|0.6154|0.5811||||||||||||0.8456|gnomADe_EAS||0&amp;1|0&amp;1||||||,A|missense_variant|MODERATE|RNF223|ENSG00000237330|Transcript|ENST00000453464|protein_coding|2/2||ENST00000453464.2:c.515C&gt;T|ENSP00000410533.1:p.Ala172Val|816|515|172|A/V|gCg/gTg|rs4333796&amp;COSV69251112||-1||SNV|HGNC|40020|YES|||||CCDS53257.1|ENSP00000410533|RN223_HUMAN||UPI0001A5E6EF|||tolerated(0.7)|benign(0)|Low_complexity_(Seg):seg&amp;PANTHER:PTHR22791&amp;PANTHER:PTHR22791:SF4|||0.5321|0.2307|0.6643|0.7917|0.5616|0.5481|0.6458|0.3916|0.7639|0.6074|0.8456|0.5287|0.6098|0.6154|0.5811||||||||||||0.8456|gnomADe_EAS||0&amp;1|0&amp;1||||||   GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,57:57:57:1740,172,0:1930:0:0  0/1:51,36:36:87:823,0,1302:1202:1734:51 0/1:38,35:35:73:854,0,946:1190:1292:38

    ```

=== &quot;Tabulé&quot;

    **Commande**
    ```
    ./vep --input_file /opt/vep/.vep/inputs/sample.vcf --output_file /opt/vep/.vep/outputs/std_output.vcf --cache --offline --format vcf --vcf
    ```

    **Output**
    ```
    ## ENSEMBL VARIANT EFFECT PREDICTOR v109.3
    ## Output produced at 2023-04-14 11:45:13
    ## Using cache in /opt/vep/.vep/homo_sapiens/109_GRCh37
    ## Using API version 109, DB version ?
    ## ensembl-funcgen version 109.cba2db8
    ## ensembl-variation version 109.18a12b6
    ## ensembl-io version 109.4946a86
    ## ensembl version 109.10baaec
    ## 1000genomes version phase3
    ## polyphen version 2.2.2
    ## dbSNP version 154
    ## assembly version GRCh37.p13
    ## COSMIC version 92
    ## genebuild version 2011-04
    ## gnomADe version r2.1
    ## regbuild version 1.0
    ## sift version sift5.2.2
    ## HGMD-PUBLIC version 20204
    ## gencode version GENCODE 19
    ## ClinVar version 202012
    ## Column descriptions:
    ## Uploaded_variation : Identifier of uploaded variant
    ## Location : Location of variant in standard coordinate format (chr:start or chr:start-end)
    ## Allele : The variant allele used to calculate the consequence
    ## Gene : Stable ID of affected gene
    ## Feature : Stable ID of feature
    ## Feature_type : Type of feature - Transcript, RegulatoryFeature or MotifFeature
    ## Consequence : Consequence type
    ## cDNA_position : Relative position of base pair in cDNA sequence
    ## CDS_position : Relative position of base pair in coding sequence
    ## Protein_position : Relative position of amino acid in protein
    ## Amino_acids : Reference and variant amino acids
    ## Codons : Reference and variant codon sequence
    ## Existing_variation : Identifier(s) of co-located known variants
    ## IMPACT : Subjective impact classification of consequence type
    ## DISTANCE : Shortest distance from variant to transcript
    ## STRAND : Strand of the feature (1/-1)
    ## FLAGS : Transcript quality flags
    ## VEP command-line: vep --cache --database 0 --dir_plugins /plugins --format vcf --input_file [PATH]/sample.vcf --offline --output_file [PATH]/std_output.csv --tab
    #Uploaded_variation Location    Allele  Gene    Feature Feature_type    Consequence cDNA_position   CDS_position    Protein_position    Amino_acids Codons  Existing_variation  IMPACT  DISTANCE    STRAND  FLAGS
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000379370 Transcript  synonymous_variant  4211    4161    1387    T   acT/acC -   LOW -   1   -
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000419249 Transcript  upstream_gene_variant   -   -   -   -   -   -   MODIFIER    1634    1   cds_start_NF,cds_end_NF
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000461111 Transcript  upstream_gene_variant   -   -   -   -   -   -   MODIFIER    3064    1   -
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000466223 Transcript  downstream_gene_variant -   -   -   -   -   -   MODIFIER    1057    1   -
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000478677 Transcript  downstream_gene_variant -   -   -   -   -   -   MODIFIER    1084    1   -
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000479707 Transcript  downstream_gene_variant -   -   -   -   -   -   MODIFIER    2573    1   -
    chr1_984302_T/C chr1:984302 C   ENSG00000188157 ENST00000492947 Transcript  non_coding_transcript_exon_variant  394 -   -   -   -   -   MODIFIER    -   1   -
    chr1_985266_C/T chr1:985266 T   ENSG00000188157 ENST00000379370 Transcript  splice_polypyrimidine_tract_variant,intron_variant  -   -   -   -   -   -   LOW -   1   -
    chr1_985266_C/T chr1:985266 T   ENSG00000188157 ENST00000419249 Transcript  upstream_gene_variant   -   -   -   -   -   -   MODIFIER    670 1   cds_start_NF,cds_end_NF
    chr1_985266_C/T chr1:985266 T   ENSG00000188157 ENST00000461111 Transcript  upstream_gene_variant   -   -   -   -   -   -   MODIFIER    2100    1   -

    ```

=== &quot;JSON&quot;

    **Commande**
    ```
    ./vep --input_file /opt/vep/.vep/inputs/sample.vcf --output_file /opt/vep/.vep/outputs/std_output.json --cache --offline --format vcf --json
    ```

    **Output**
    ```json
    {&quot;allele_string&quot;:&quot;T/C&quot;,&quot;most_severe_consequence&quot;:&quot;synonymous_variant&quot;,&quot;id&quot;:&quot;.&quot;,&quot;transcript_consequences&quot;:[{&quot;variant_allele&quot;:&quot;C&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;impact&quot;:&quot;LOW&quot;,&quot;cds_end&quot;:4161,&quot;transcript_id&quot;:&quot;ENST00000379370&quot;,&quot;strand&quot;:1,&quot;amino_acids&quot;:&quot;T&quot;,&quot;cds_start&quot;:4161,&quot;cdna_end&quot;:4211,&quot;codons&quot;:&quot;acT/acC&quot;,&quot;protein_end&quot;:1387,&quot;cdna_start&quot;:4211,&quot;protein_start&quot;:1387,&quot;consequence_terms&quot;:[&quot;synonymous_variant&quot;]},{&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;C&quot;,&quot;flags&quot;:[&quot;cds_start_NF&quot;,&quot;cds_end_NF&quot;],&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000419249&quot;,&quot;distance&quot;:1634,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;]},{&quot;transcript_id&quot;:&quot;ENST00000461111&quot;,&quot;strand&quot;:1,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;variant_allele&quot;:&quot;C&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;],&quot;distance&quot;:3064},{&quot;distance&quot;:1057,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;variant_allele&quot;:&quot;C&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;transcript_id&quot;:&quot;ENST00000466223&quot;,&quot;strand&quot;:1},{&quot;distance&quot;:1084,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;variant_allele&quot;:&quot;C&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;transcript_id&quot;:&quot;ENST00000478677&quot;,&quot;strand&quot;:1},{&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;distance&quot;:2573,&quot;transcript_id&quot;:&quot;ENST00000479707&quot;,&quot;strand&quot;:1,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;variant_allele&quot;:&quot;C&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;},{&quot;cdna_end&quot;:394,&quot;cdna_start&quot;:394,&quot;consequence_terms&quot;:[&quot;non_coding_transcript_exon_variant&quot;],&quot;variant_allele&quot;:&quot;C&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;transcript_id&quot;:&quot;ENST00000492947&quot;,&quot;strand&quot;:1}],&quot;end&quot;:984302,&quot;input&quot;:&quot;chr1\t984302\t.\tT\tC\t3877.41\tPASS\tAB=0.464088;ABP=5.03781;AC=4;AF=0.667;AN=6;AO=151;CIGAR=1X;DP=249\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:1,67:67:68:1979,171,0:2234:34:1\t0/1:52,35:35:87:812,0,1296:1190:1728:52\t0/1:45,49:49:94:1219,0,1056:1666:1484:45&quot;,&quot;seq_region_name&quot;:&quot;chr1&quot;,&quot;start&quot;:984302,&quot;strand&quot;:1,&quot;assembly_name&quot;:&quot;GRCh37&quot;}
    {&quot;seq_region_name&quot;:&quot;chr1&quot;,&quot;input&quot;:&quot;chr1\t985266\t.\tC\tT\t2801.75\tPASS\tAB=0.491667;ABP=3.08268;AC=4;AF=0.667;AN=6;AO=110;CIGAR=1X;DP=171\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,51:51:51:1549,154,0:1718:0:0\t0/1:33,27:27:60:649,0,825:918:1114:33\t0/1:28,32:32:60:782,0,679:1066:952:28&quot;,&quot;end&quot;:985266,&quot;transcript_consequences&quot;:[{&quot;strand&quot;:1,&quot;consequence_terms&quot;:[&quot;splice_polypyrimidine_tract_variant&quot;,&quot;intron_variant&quot;],&quot;transcript_id&quot;:&quot;ENST00000379370&quot;,&quot;impact&quot;:&quot;LOW&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;},{&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000419249&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;flags&quot;:[&quot;cds_start_NF&quot;,&quot;cds_end_NF&quot;],&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;],&quot;distance&quot;:670},{&quot;distance&quot;:2100,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;],&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000461111&quot;},{&quot;distance&quot;:2021,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000466223&quot;},{&quot;transcript_id&quot;:&quot;ENST00000478677&quot;,&quot;strand&quot;:1,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;distance&quot;:2048},{&quot;distance&quot;:3537,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000479707&quot;},{&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000492947&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;distance&quot;:492}],&quot;id&quot;:&quot;.&quot;,&quot;most_severe_consequence&quot;:&quot;splice_polypyrimidine_tract_variant&quot;,&quot;allele_string&quot;:&quot;C/T&quot;,&quot;assembly_name&quot;:&quot;GRCh37&quot;,&quot;strand&quot;:1,&quot;start&quot;:985266}
    {&quot;start&quot;:986732,&quot;strand&quot;:1,&quot;assembly_name&quot;:&quot;GRCh37&quot;,&quot;id&quot;:&quot;.&quot;,&quot;transcript_consequences&quot;:[{&quot;protein_start&quot;:1785,&quot;consequence_terms&quot;:[&quot;missense_variant&quot;],&quot;cdna_start&quot;:5403,&quot;protein_end&quot;:1785,&quot;codons&quot;:&quot;Gac/Aac&quot;,&quot;cdna_end&quot;:5403,&quot;transcript_id&quot;:&quot;ENST00000379370&quot;,&quot;amino_acids&quot;:&quot;D/N&quot;,&quot;strand&quot;:1,&quot;cds_start&quot;:5353,&quot;impact&quot;:&quot;MODERATE&quot;,&quot;cds_end&quot;:5353,&quot;variant_allele&quot;:&quot;A&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;},{&quot;distance&quot;:3681,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;],&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;gene_id&quot;:&quot;ENSG00000242590&quot;,&quot;variant_allele&quot;:&quot;A&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000418300&quot;},{&quot;consequence_terms&quot;:[&quot;missense_variant&quot;],&quot;cdna_start&quot;:260,&quot;protein_end&quot;:88,&quot;transcript_id&quot;:&quot;ENST00000419249&quot;,&quot;cds_end&quot;:262,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;A&quot;,&quot;flags&quot;:[&quot;cds_start_NF&quot;,&quot;cds_end_NF&quot;],&quot;protein_start&quot;:88,&quot;codons&quot;:&quot;Gac/Aac&quot;,&quot;cdna_end&quot;:260,&quot;amino_acids&quot;:&quot;D/N&quot;,&quot;strand&quot;:1,&quot;cds_start&quot;:262,&quot;impact&quot;:&quot;MODERATE&quot;},{&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000461111&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;A&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;],&quot;distance&quot;:634},{&quot;distance&quot;:3487,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;A&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000466223&quot;},{&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;variant_allele&quot;:&quot;A&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;transcript_id&quot;:&quot;ENST00000478677&quot;,&quot;strand&quot;:1,&quot;distance&quot;:3514,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;]},{&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;distance&quot;:1958,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000492947&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;A&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;}],&quot;most_severe_consequence&quot;:&quot;missense_variant&quot;,&quot;allele_string&quot;:&quot;G/A&quot;,&quot;seq_region_name&quot;:&quot;chr1&quot;,&quot;input&quot;:&quot;chr1\t986732\t.\tG\tA\t922.52\tPASS\tAB=0.415094;ABP=9.64763;AC=1;AF=0.167;AN=6;AO=44;CIGAR=1X;DP=356\tGT:AD:AO:DP:PL:QA:QR:RO\t0/0:129,0:0:129:0,388,3908:0:4340:129\t0/1:62,44:44:106:1030,0,1581:1496:2108:62\t0/0:121,0:0:121:0,364,3683:0:4090:121&quot;,&quot;end&quot;:986732}
    {&quot;assembly_name&quot;:&quot;GRCh37&quot;,&quot;start&quot;:987200,&quot;strand&quot;:1,&quot;seq_region_name&quot;:&quot;chr1&quot;,&quot;end&quot;:987200,&quot;input&quot;:&quot;chr1\t987200\t.\tC\tT\t6412.21\tPASS\tAB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=219;CIGAR=1X;DP=219\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,63:63:63:1911,190,0:2120:0:0\t1/1:0,66:66:66:1963,199,0:2178:0:0\t1/1:0,90:90:90:2737,271,0:3038:0:0&quot;,&quot;id&quot;:&quot;.&quot;,&quot;transcript_consequences&quot;:[{&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;impact&quot;:&quot;LOW&quot;,&quot;consequence_terms&quot;:[&quot;splice_donor_5th_base_variant&quot;,&quot;intron_variant&quot;],&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000379370&quot;},{&quot;transcript_id&quot;:&quot;ENST00000418300&quot;,&quot;strand&quot;:1,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;gene_id&quot;:&quot;ENSG00000242590&quot;,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;],&quot;distance&quot;:3213},{&quot;transcript_id&quot;:&quot;ENST00000419249&quot;,&quot;strand&quot;:1,&quot;variant_allele&quot;:&quot;T&quot;,&quot;flags&quot;:[&quot;cds_start_NF&quot;,&quot;cds_end_NF&quot;],&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;impact&quot;:&quot;LOW&quot;,&quot;consequence_terms&quot;:[&quot;splice_donor_5th_base_variant&quot;,&quot;intron_variant&quot;]},{&quot;distance&quot;:166,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;],&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000461111&quot;},{&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;distance&quot;:3955,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000466223&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;},{&quot;distance&quot;:3982,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000478677&quot;},{&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;distance&quot;:2426,&quot;transcript_id&quot;:&quot;ENST00000492947&quot;,&quot;strand&quot;:1,&quot;variant_allele&quot;:&quot;T&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;}],&quot;allele_string&quot;:&quot;C/T&quot;,&quot;most_severe_consequence&quot;:&quot;splice_donor_5th_base_variant&quot;}
    {&quot;end&quot;:990280,&quot;input&quot;:&quot;chr1\t990280\t.\tC\tT\t4547.65\tPASS\tAB=0.446602;ABP=8.11221;AC=4;AF=0.667;AN=6;AO=175;CIGAR=1X;DP=289\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,83:83:83:2496,250,0:2770:0:0\t0/1:56,39:39:95:910,0,1371:1326:1838:56\t0/1:58,53:53:111:1283,0,1436:1794:1964:58&quot;,&quot;seq_region_name&quot;:&quot;chr1&quot;,&quot;most_severe_consequence&quot;:&quot;synonymous_variant&quot;,&quot;allele_string&quot;:&quot;C/T&quot;,&quot;id&quot;:&quot;.&quot;,&quot;transcript_consequences&quot;:[{&quot;cdna_end&quot;:6107,&quot;codons&quot;:&quot;gaC/gaT&quot;,&quot;protein_end&quot;:2019,&quot;cdna_start&quot;:6107,&quot;protein_start&quot;:2019,&quot;consequence_terms&quot;:[&quot;synonymous_variant&quot;],&quot;variant_allele&quot;:&quot;T&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;impact&quot;:&quot;LOW&quot;,&quot;cds_end&quot;:6057,&quot;transcript_id&quot;:&quot;ENST00000379370&quot;,&quot;cds_start&quot;:6057,&quot;strand&quot;:1,&quot;amino_acids&quot;:&quot;D&quot;},{&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000418300&quot;,&quot;gene_id&quot;:&quot;ENSG00000242590&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;],&quot;distance&quot;:133},{&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;flags&quot;:[&quot;cds_start_NF&quot;,&quot;cds_end_NF&quot;],&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000419249&quot;,&quot;distance&quot;:950,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;]},{&quot;distance&quot;:4834,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;],&quot;gene_id&quot;:&quot;ENSG00000217801&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000427998&quot;},{&quot;cdna_end&quot;:2173,&quot;cdna_start&quot;:2173,&quot;consequence_terms&quot;:[&quot;non_coding_transcript_exon_variant&quot;],&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000461111&quot;}],&quot;assembly_name&quot;:&quot;GRCh37&quot;,&quot;start&quot;:990280,&quot;strand&quot;:1}
    {&quot;strand&quot;:1,&quot;start&quot;:990380,&quot;assembly_name&quot;:&quot;GRCh37&quot;,&quot;transcript_consequences&quot;:[{&quot;transcript_id&quot;:&quot;ENST00000379370&quot;,&quot;strand&quot;:1,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;consequence_terms&quot;:[&quot;3_prime_UTR_variant&quot;],&quot;cdna_start&quot;:6207,&quot;cdna_end&quot;:6207},{&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;gene_id&quot;:&quot;ENSG00000242590&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000418300&quot;,&quot;distance&quot;:33,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;]},{&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;flags&quot;:[&quot;cds_start_NF&quot;,&quot;cds_end_NF&quot;],&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000419249&quot;,&quot;distance&quot;:1050,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;]},{&quot;transcript_id&quot;:&quot;ENST00000427998&quot;,&quot;strand&quot;:1,&quot;variant_allele&quot;:&quot;T&quot;,&quot;gene_id&quot;:&quot;ENSG00000217801&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;],&quot;distance&quot;:4734},{&quot;cdna_start&quot;:2273,&quot;consequence_terms&quot;:[&quot;non_coding_transcript_exon_variant&quot;],&quot;cdna_end&quot;:2273,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000461111&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;}],&quot;id&quot;:&quot;.&quot;,&quot;allele_string&quot;:&quot;C/T&quot;,&quot;most_severe_consequence&quot;:&quot;3_prime_UTR_variant&quot;,&quot;seq_region_name&quot;:&quot;chr1&quot;,&quot;end&quot;:990380,&quot;input&quot;:&quot;chr1\t990380\t.\tC\tT\t570.29\tPASS\tAB=0.55102;ABP=4.11819;AC=2;AF=0.333;AN=6;AO=27;CIGAR=1X;DP=63\tGT:AD:AO:DP:PL:QA:QR:RO\t0/0:14,0:0:14:0,42,432:0:476:14\t0/1:8,14:14:22:365,0,182:476:272:8\t0/1:14,13:13:27:320,0,343:442:468:14&quot;}
    {&quot;input&quot;:&quot;chr1\t990417\t.\tT\tC\t328.36\tPASS\tAB=0.580645;ABP=4.76149;AC=2;AF=0.333;AN=6;AO=18;CIGAR=1X;DP=41\tGT:AD:AO:DP:PL:QA:QR:RO\t0/0:10,0:0:10:0,30,309:0:340:10\t0/1:5,9:9:14:237,0,114:306:170:5\t0/1:8,9:9:17:220,0,197:298:272:8&quot;,&quot;end&quot;:990417,&quot;seq_region_name&quot;:&quot;chr1&quot;,&quot;allele_string&quot;:&quot;T/C&quot;,&quot;most_severe_consequence&quot;:&quot;3_prime_UTR_variant&quot;,&quot;id&quot;:&quot;.&quot;,&quot;transcript_consequences&quot;:[{&quot;cdna_start&quot;:6244,&quot;consequence_terms&quot;:[&quot;3_prime_UTR_variant&quot;],&quot;cdna_end&quot;:6244,&quot;transcript_id&quot;:&quot;ENST00000379370&quot;,&quot;strand&quot;:1,&quot;variant_allele&quot;:&quot;C&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;},{&quot;cdna_end&quot;:5,&quot;cdna_start&quot;:5,&quot;consequence_terms&quot;:[&quot;non_coding_transcript_exon_variant&quot;],&quot;variant_allele&quot;:&quot;C&quot;,&quot;gene_id&quot;:&quot;ENSG00000242590&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;transcript_id&quot;:&quot;ENST00000418300&quot;,&quot;strand&quot;:1},{&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;C&quot;,&quot;flags&quot;:[&quot;cds_start_NF&quot;,&quot;cds_end_NF&quot;],&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000419249&quot;,&quot;distance&quot;:1087,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;]},{&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;],&quot;distance&quot;:4697,&quot;transcript_id&quot;:&quot;ENST00000427998&quot;,&quot;strand&quot;:1,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;variant_allele&quot;:&quot;C&quot;,&quot;gene_id&quot;:&quot;ENSG00000217801&quot;},{&quot;cdna_end&quot;:2310,&quot;cdna_start&quot;:2310,&quot;consequence_terms&quot;:[&quot;non_coding_transcript_exon_variant&quot;],&quot;variant_allele&quot;:&quot;C&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;transcript_id&quot;:&quot;ENST00000461111&quot;,&quot;strand&quot;:1}],&quot;assembly_name&quot;:&quot;GRCh37&quot;,&quot;start&quot;:990417,&quot;strand&quot;:1}
    {&quot;most_severe_consequence&quot;:&quot;3_prime_UTR_variant&quot;,&quot;allele_string&quot;:&quot;C/T&quot;,&quot;transcript_consequences&quot;:[{&quot;cdna_start&quot;:6344,&quot;consequence_terms&quot;:[&quot;3_prime_UTR_variant&quot;],&quot;cdna_end&quot;:6344,&quot;transcript_id&quot;:&quot;ENST00000379370&quot;,&quot;strand&quot;:1,&quot;variant_allele&quot;:&quot;T&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;},{&quot;consequence_terms&quot;:[&quot;non_coding_transcript_exon_variant&quot;],&quot;cdna_start&quot;:105,&quot;cdna_end&quot;:105,&quot;transcript_id&quot;:&quot;ENST00000418300&quot;,&quot;strand&quot;:1,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;gene_id&quot;:&quot;ENSG00000242590&quot;},{&quot;distance&quot;:1187,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;flags&quot;:[&quot;cds_start_NF&quot;,&quot;cds_end_NF&quot;],&quot;variant_allele&quot;:&quot;T&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;transcript_id&quot;:&quot;ENST00000419249&quot;,&quot;strand&quot;:1},{&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000427998&quot;,&quot;gene_id&quot;:&quot;ENSG00000217801&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;],&quot;distance&quot;:4597},{&quot;cdna_end&quot;:2410,&quot;cdna_start&quot;:2410,&quot;consequence_terms&quot;:[&quot;non_coding_transcript_exon_variant&quot;],&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000461111&quot;}],&quot;id&quot;:&quot;.&quot;,&quot;input&quot;:&quot;chr1\t990517\t.\tC\tT\t182.67\tSNP_filter\tAB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=8;CIGAR=1X;DP=8\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,1:1:1:34,3,0:34:0:0\t1/1:0,4:4:4:126,12,0:136:0:0\t1/1:0,3:3:3:95,9,0:102:0:0&quot;,&quot;end&quot;:990517,&quot;seq_region_name&quot;:&quot;chr1&quot;,&quot;strand&quot;:1,&quot;start&quot;:990517,&quot;assembly_name&quot;:&quot;GRCh37&quot;}
    {&quot;seq_region_name&quot;:&quot;chr1&quot;,&quot;input&quot;:&quot;chr1\t990984\t.\tG\tA\t82.39\tLowCoverage;SNP_filter\tAB=0;ABP=0;AC=2;AF=1.00;AN=2;AO=3;CIGAR=1X;DP=3\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,3:3:3:95,9,0:102:0:0\t.\t.&quot;,&quot;end&quot;:990984,&quot;transcript_consequences&quot;:[{&quot;consequence_terms&quot;:[&quot;3_prime_UTR_variant&quot;],&quot;cdna_start&quot;:6811,&quot;cdna_end&quot;:6811,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000379370&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;A&quot;},{&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;gene_id&quot;:&quot;ENSG00000242590&quot;,&quot;variant_allele&quot;:&quot;A&quot;,&quot;consequence_terms&quot;:[&quot;intron_variant&quot;,&quot;non_coding_transcript_variant&quot;],&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000418300&quot;},{&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;variant_allele&quot;:&quot;A&quot;,&quot;flags&quot;:[&quot;cds_start_NF&quot;,&quot;cds_end_NF&quot;],&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;transcript_id&quot;:&quot;ENST00000419249&quot;,&quot;strand&quot;:1,&quot;distance&quot;:1654,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;]},{&quot;gene_id&quot;:&quot;ENSG00000217801&quot;,&quot;variant_allele&quot;:&quot;A&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000427998&quot;,&quot;distance&quot;:4130,&quot;consequence_terms&quot;:[&quot;upstream_gene_variant&quot;]},{&quot;consequence_terms&quot;:[&quot;non_coding_transcript_exon_variant&quot;],&quot;cdna_start&quot;:2877,&quot;cdna_end&quot;:2877,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000461111&quot;,&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;gene_id&quot;:&quot;ENSG00000188157&quot;,&quot;variant_allele&quot;:&quot;A&quot;}],&quot;id&quot;:&quot;.&quot;,&quot;most_severe_consequence&quot;:&quot;3_prime_UTR_variant&quot;,&quot;allele_string&quot;:&quot;G/A&quot;,&quot;assembly_name&quot;:&quot;GRCh37&quot;,&quot;strand&quot;:1,&quot;start&quot;:990984}
    {&quot;assembly_name&quot;:&quot;GRCh37&quot;,&quot;strand&quot;:1,&quot;start&quot;:1007203,&quot;input&quot;:&quot;chr1\t1007203\t.\tA\tG\t3110.06\tPASS\tAB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=105;CIGAR=1X;DP=105\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,26:26:26:752,78,0:832:0:0\t1/1:0,46:46:46:1411,138,0:1564:0:0\t1/1:0,33:33:33:1013,99,0:1122:0:0&quot;,&quot;end&quot;:1007203,&quot;seq_region_name&quot;:&quot;chr1&quot;,&quot;allele_string&quot;:&quot;A/G&quot;,&quot;most_severe_consequence&quot;:&quot;synonymous_variant&quot;,&quot;transcript_consequences&quot;:[{&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;variant_allele&quot;:&quot;G&quot;,&quot;gene_id&quot;:&quot;ENSG00000217801&quot;,&quot;transcript_id&quot;:&quot;ENST00000412397&quot;,&quot;strand&quot;:1,&quot;distance&quot;:2468,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;]},{&quot;cds_start&quot;:744,&quot;amino_acids&quot;:&quot;D&quot;,&quot;strand&quot;:-1,&quot;transcript_id&quot;:&quot;ENST00000453464&quot;,&quot;cds_end&quot;:744,&quot;impact&quot;:&quot;LOW&quot;,&quot;gene_id&quot;:&quot;ENSG00000237330&quot;,&quot;variant_allele&quot;:&quot;G&quot;,&quot;protein_start&quot;:248,&quot;consequence_terms&quot;:[&quot;synonymous_variant&quot;],&quot;cdna_start&quot;:1045,&quot;codons&quot;:&quot;gaT/gaC&quot;,&quot;protein_end&quot;:248,&quot;cdna_end&quot;:1045}],&quot;id&quot;:&quot;.&quot;}
    {&quot;assembly_name&quot;:&quot;GRCh37&quot;,&quot;strand&quot;:1,&quot;start&quot;:1007222,&quot;seq_region_name&quot;:&quot;chr1&quot;,&quot;end&quot;:1007222,&quot;input&quot;:&quot;chr1\t1007222\t.\tG\tT\t1780.81\tPASS\tAB=0.685185;ABP=35.1803;AC=2;AF=0.333;AN=6;AO=74;CIGAR=1X;DP=143\tGT:AD:AO:DP:PL:QA:QR:RO\t0/0:35,0:0:35:0,105,1034:0:1146:35\t0/1:16,40:40:56:1044,0,277:1344:492:16\t0/1:18,34:34:52:887,0,397:1156:612:18&quot;,&quot;transcript_consequences&quot;:[{&quot;distance&quot;:2487,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;gene_id&quot;:&quot;ENSG00000217801&quot;,&quot;transcript_id&quot;:&quot;ENST00000412397&quot;,&quot;strand&quot;:1},{&quot;cdna_end&quot;:1026,&quot;protein_end&quot;:242,&quot;codons&quot;:&quot;cCc/cAc&quot;,&quot;cdna_start&quot;:1026,&quot;protein_start&quot;:242,&quot;consequence_terms&quot;:[&quot;missense_variant&quot;],&quot;gene_id&quot;:&quot;ENSG00000237330&quot;,&quot;variant_allele&quot;:&quot;T&quot;,&quot;cds_end&quot;:725,&quot;impact&quot;:&quot;MODERATE&quot;,&quot;cds_start&quot;:725,&quot;amino_acids&quot;:&quot;P/H&quot;,&quot;strand&quot;:-1,&quot;transcript_id&quot;:&quot;ENST00000453464&quot;}],&quot;id&quot;:&quot;.&quot;,&quot;allele_string&quot;:&quot;G/T&quot;,&quot;most_severe_consequence&quot;:&quot;missense_variant&quot;}
    {&quot;strand&quot;:1,&quot;start&quot;:1007432,&quot;assembly_name&quot;:&quot;GRCh37&quot;,&quot;most_severe_consequence&quot;:&quot;missense_variant&quot;,&quot;allele_string&quot;:&quot;G/A&quot;,&quot;transcript_consequences&quot;:[{&quot;distance&quot;:2697,&quot;consequence_terms&quot;:[&quot;downstream_gene_variant&quot;],&quot;impact&quot;:&quot;MODIFIER&quot;,&quot;gene_id&quot;:&quot;ENSG00000217801&quot;,&quot;variant_allele&quot;:&quot;A&quot;,&quot;strand&quot;:1,&quot;transcript_id&quot;:&quot;ENST00000412397&quot;},{&quot;cds_end&quot;:515,&quot;impact&quot;:&quot;MODERATE&quot;,&quot;gene_id&quot;:&quot;ENSG00000237330&quot;,&quot;variant_allele&quot;:&quot;A&quot;,&quot;strand&quot;:-1,&quot;cds_start&quot;:515,&quot;amino_acids&quot;:&quot;A/V&quot;,&quot;transcript_id&quot;:&quot;ENST00000453464&quot;,&quot;codons&quot;:&quot;gCg/gTg&quot;,&quot;protein_end&quot;:172,&quot;cdna_end&quot;:816,&quot;consequence_terms&quot;:[&quot;missense_variant&quot;],&quot;protein_start&quot;:172,&quot;cdna_start&quot;:816}],&quot;id&quot;:&quot;.&quot;,&quot;input&quot;:&quot;chr1\t1007432\t.\tG\tA\t3295.23\tPASS\tAB=0.44375;ABP=7.40753;AC=4;AF=0.667;AN=6;AO=128;CIGAR=1X;DP=217\tGT:AD:AO:DP:PL:QA:QR:RO\t1/1:0,57:57:57:1740,172,0:1930:0:0\t0/1:51,36:36:87:823,0,1302:1202:1734:51\t0/1:38,35:35:73:854,0,946:1190:1292:38&quot;,&quot;end&quot;:1007432,&quot;seq_region_name&quot;:&quot;chr1&quot;}
    ```
</code></pre></div></td></tr></table></div>
<h3 id="annotations-avancees">Annotations avancées</h3>
<p>Il est possible de compléter les annotations de base de VEP par des annotations issues de bases de données prévues à cet effet (plugins) ou créées par l'utilisateur (custom).</p>
<h4 id="plugins">Plugins</h4>
<p>Les <a href="https://www.ensembl.org/info/docs/tools/vep/script/vep_plugins.html">plugins</a> sont des extensions de VEP nécessitant une installation (simplement avec la commande <code>perl INSTALL.pl -a p --PLUGINS dbNSFP,CADD</code>). Une fois l'installation faite et la source d'annotations récupérée, il suffit pour ajouter ces informations d'appeler le plugin lors du lancement d'une annotation, préciser le chemin de la source d'annotation et selon les plugins, préciser les colonnes désirées.</p>
<p>Par exemple : 
<div class="highlight"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre><span></span><span class="normal">1</span>
<span class="normal">2</span></pre></div></td><td class="code"><div><pre><span></span><code>./vep --input_file /opt/vep/.vep/inputs/sample.vcf --output_file /opt/vep/.vep/outputs/std_output.vcf --cache --offline --everything --format vcf --vcf \
<span class="hll">--plugin dbNSFP,path/to/file.gz,CADD_raw_hg19,REVEL_score,Polyphen2_HVAR_score
</span></code></pre></div></td></tr></table></div></p>
<h4 id="custom">Custom</h4>
<p>Il est également possible d'ajouter des annotations de fichiers BED, VCF, GFF, GTF ou bigWig tant qu'ils ont été préalablement indexés via <a href="https://www.htslib.org/doc/tabix.html">tabix</a>.<br />
Il suffit ensuite de préciser au lancement d'une annotation le chemin vers le fichier, le nom pour nommer les colonnes, le format de fichier, le type d'annotation, s'il faut rapporter les coordonnées de la feature recouvrante puis les champs à récupérer.</p>
<p>Par exemple :
<div class="highlight"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre><span></span><span class="normal">1</span>
<span class="normal">2</span></pre></div></td><td class="code"><div><pre><span></span><code>./vep --input_file /opt/vep/.vep/inputs/sample.vcf --output_file /opt/vep/.vep/outputs/std_output.vcf --cache --offline --everything --format vcf --vcf \
<span class="hll">--custom [PATH]/clinvar_20230326.vcf.gz,ClinVar,vcf,exact,0,CLNDN,CLNSIG,CLNREVSTAT
</span></code></pre></div></td></tr></table></div></p>
<h3 id="precision-sur-lannotation-de-vcf">Précision sur l'annotation de VCF</h3>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>Il est préférable de bien connaître les spécificités du <a href="../vcf">format VCF</a></p>
</div>
<p>Voici un exemple de VCF réunissant des colonnes de plugins et de la base ClinVar la plus récente, mais n'affichant que le transcrit considéré prioritaire pour plus de lisibilité.</p>
<p>??? abstract "Exemple de fichier VCF annoté par VEP"</p>
<div class="highlight"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre><span></span><span class="normal"> 1</span>
<span class="normal"> 2</span>
<span class="normal"> 3</span>
<span class="normal"> 4</span>
<span class="normal"> 5</span>
<span class="normal"> 6</span>
<span class="normal"> 7</span>
<span class="normal"> 8</span>
<span class="normal"> 9</span>
<span class="normal">10</span>
<span class="normal">11</span>
<span class="normal">12</span>
<span class="normal">13</span>
<span class="normal">14</span>
<span class="normal">15</span>
<span class="normal">16</span>
<span class="normal">17</span>
<span class="normal">18</span>
<span class="normal">19</span>
<span class="normal">20</span>
<span class="normal">21</span>
<span class="normal">22</span>
<span class="normal">23</span>
<span class="normal">24</span>
<span class="normal">25</span>
<span class="normal">26</span>
<span class="normal">27</span>
<span class="normal">28</span>
<span class="normal">29</span>
<span class="normal">30</span>
<span class="normal">31</span>
<span class="normal">32</span>
<span class="normal">33</span>
<span class="normal">34</span>
<span class="normal">35</span>
<span class="normal">36</span>
<span class="normal">37</span>
<span class="normal">38</span>
<span class="normal">39</span>
<span class="normal">40</span>
<span class="normal">41</span>
<span class="normal">42</span></pre></div></td><td class="code"><div><pre><span></span><code>``` hl_lines=&quot;3 4 27&quot;
[...]
##INFO=&lt;ID=DP,Number=1,Type=Integer,Description=&quot;Approximate read depth; some reads may have been filtered&quot;&gt;
##VEP=&quot;v109&quot; time=&quot;2023-04-12 19:40:52&quot; cache=&quot;/opt/vep/.vep/homo_sapiens/109_GRCh37&quot; ensembl-funcgen=109.cba2db8 ensembl-variation=109.18a12b6 ensembl-io=109.4946a86 ensembl=109.10baaec 1000genomes=&quot;phase3&quot; COSMIC=&quot;92&quot; ClinVar=&quot;202012&quot; HGMD-PUBLIC=&quot;20204&quot; assembly=&quot;GRCh37.p13&quot; dbSNP=&quot;154&quot; gencode=&quot;GENCODE 19&quot; genebuild=&quot;2011-04&quot; gnomADe=&quot;r2.1&quot; polyphen=&quot;2.2.2&quot; regbuild=&quot;1.0&quot; sift=&quot;sift5.2.2&quot;
##INFO=&lt;ID=CSQ,Number=.,Type=String,Description=&quot;Consequence annotations from Ensembl VEP. Format: Gene|Feature|Feature_Type|HGVSg|HGVSc|HGVSp|Consequence|IMPACT|VARIANT_CLASS|Codons|Amino_acids|cDNA_position|CDS_position|ClinVar_CLNDN|ClinVar_CLNSIG|ClinVar_CLNREVSTAT|LOEUF|CADD_raw_hg19|GERP++_RS|REVEL_score|Polyphen2_HVAR|gnomAD_genomes_AC|gnomAD_genomes_AF|gnomAD_genomes_nhomalt|gnomAD_genomes_POPMAX_AF|SpliceAI_pred_DP_AG|SpliceAI_pred_DP_AL|SpliceAI_pred_DP_DG|SpliceAI_pred_DP_DL|SpliceAI_pred_DS_AG|SpliceAI_pred_DS_AL|SpliceAI_pred_DS_DG|SpliceAI_pred_DS_DL&quot;&gt;
##CADD_raw_hg19=(from dbNSFP4.3a_grch37) CADD raw score for functional prediction of a SNP using the hg19 model. Please refer to Kircher et al. (2014) Nature Genetics 46(3):310-5 for details. The larger the score the more likely the SNP has damaging effect. Scores range from -6.458163 to 18.301497 in dbNSFP. Please note the following copyright statement for CADD: &quot;CADD scores (http://cadd.gs.washington.edu/) are Copyright 2013 University of Washington and Hudson-Alpha Institute for Biotechnology (all rights reserved) but are freely available for all academic, non-commercial applications. For commercial licensing information contact Jennifer McCullar (mccullaj@uw.edu).&quot;
##GERP++_RS=(from dbNSFP4.3a_grch37) GERP++ RS score, the larger the score, the more conserved the site. Scores range from -12.3 to 6.17.
##Polyphen2_HVAR_score=(from dbNSFP4.3a_grch37) Polyphen2 score based on HumVar, i.e. hvar_prob. The score ranges from 0 to 1. Multiple entries separated by &quot;;&quot;, corresponding to Uniprot_acc.
##REVEL_score=(from dbNSFP4.3a_grch37) REVEL is an ensemble score based on 13 individual scores for predicting the pathogenicity of missense variants. Scores range from 0 to 1. The larger the score the more likely the SNP has damaging effect. &quot;REVEL scores are freely available for non-commercial use. For other uses, please contact Weiva Sieh&quot; (weiva.sieh@mssm.edu) Multiple entries are separated by &quot;;&quot;, corresponding to Ensembl_transcriptid.
##gnomAD_genomes_AC=(from dbNSFP4.3a_grch37) Alternative allele count in the whole gnomAD genome samples v3.1 For mtDNA, this is sum of AC_hom (&quot;Allele count restricted to variants with a heteroplasmy level &gt;= 0.95&quot;) and AC_het (&quot;Allele count restricted to variants with a heteroplasmy level &gt;= 0.10 and &lt; 0.95&quot;)
##gnomAD_genomes_AF=(from dbNSFP4.3a_grch37) Alternative allele frequency in the whole gnomAD genome samples v3.1 For mtDNA, this is sum of AF_hom (&quot;Allele frequency restricted to variants with a heteroplasmy level &gt;= 0.95&quot;) and AF_het (&quot;Allele frequency restricted to variants with a heteroplasmy level &gt;= 0.10 and &lt; 0.95&quot;)
##gnomAD_genomes_POPMAX_AF=(from dbNSFP4.3a_grch37) Maximum allele frequency across populations (excluding samples of Ashkenazi, Finnish, and indeterminate ancestry)
##gnomAD_genomes_nhomalt=(from dbNSFP4.3a_grch37) Count of individuals with homozygous alternative allele in the whole gnomAD genome samples v3.1 For mtDNA, this is AC_hom (&quot;Allele count restricted to variants with a heteroplasmy level &gt;= 0.95&quot;)
##SpliceAI_pred_DP_AG=SpliceAI predicted effect on splicing. Delta position for acceptor gain
##SpliceAI_pred_DP_AL=SpliceAI predicted effect on splicing. Delta position for acceptor loss
##SpliceAI_pred_DP_DG=SpliceAI predicted effect on splicing. Delta position for donor gain
##SpliceAI_pred_DP_DL=SpliceAI predicted effect on splicing. Delta position for donor loss
##SpliceAI_pred_DS_AG=SpliceAI predicted effect on splicing. Delta score for acceptor gain
##SpliceAI_pred_DS_AL=SpliceAI predicted effect on splicing. Delta score for acceptor loss
##SpliceAI_pred_DS_DG=SpliceAI predicted effect on splicing. Delta score for donor gain
##SpliceAI_pred_DS_DL=SpliceAI predicted effect on splicing. Delta score for donor loss
##SpliceAI_pred_SYMBOL=SpliceAI gene symbol
##LOEUF=Loss-of-function observed/expected upper bound fraction
##INFO=&lt;ID=ClinVar,Number=.,Type=String,Description=&quot;/opt/vep/.vep/hg19/clinvar/clinvar_20230326.vcf.gz&quot;&gt;
##INFO=&lt;ID=ClinVar_CLNDN,Number=.,Type=String,Description=&quot;CLNDN field from /opt/vep/.vep/hg19/clinvar/clinvar_20230326.vcf.gz&quot;&gt;
##INFO=&lt;ID=ClinVar_CLNSIG,Number=.,Type=String,Description=&quot;CLNSIG field from /opt/vep/.vep/hg19/clinvar/clinvar_20230326.vcf.gz&quot;&gt;
##INFO=&lt;ID=ClinVar_CLNREVSTAT,Number=.,Type=String,Description=&quot;CLNREVSTAT field from /opt/vep/.vep/hg19/clinvar/clinvar_20230326.vcf.gz&quot;&gt;
##VEP-command-line=&#39;vep --cache --custom [PATH]/clinvar_20230326.vcf.gz,ClinVar,vcf,exact,0,CLNDN,CLNSIG,CLNREVSTAT --database 0 --dir_plugins /plugins --fields Gene,Feature,Feature_Type,HGVSg,HGVSc,HGVSp,Consequence,IMPACT,VARIANT_CLASS,Codons,Amino_acids,cDNA_position,CDS_position,ClinVar_CLNDN,ClinVar_CLNSIG,ClinVar_CLNREVSTAT,LOEUF,CADD_raw_hg19,GERP++_RS,REVEL_score,Polyphen2_HVAR,gnomAD_genomes_AC,gnomAD_genomes_AF,gnomAD_genomes_nhomalt,gnomAD_genomes_POPMAX_AF,SpliceAI_pred_DP_AG,SpliceAI_pred_DP_AL,SpliceAI_pred_DP_DG,SpliceAI_pred_DP_DL,SpliceAI_pred_DS_AG,SpliceAI_pred_DS_AL,SpliceAI_pred_DS_DG,SpliceAI_pred_DS_DL --force_overwrite --fork 6 --format vcf --hgvs --input_file [PATH]/sample.vcf --no_stats --offline --output_file [PATH]/example_pick.vcf --pick --plugin [PATH]/loeuf_dataset.tsv.gz,match_by=transcript --vcf&#39;
#CHROM  POS ID  REF ALT QUAL    FILTER  INFO    FORMAT  22-23121-A-02-00    22-23123-A-02-00    22-24543-A-01-00
chr1    984302  .   T   C   3877.41 PASS    AB=0.464088;ABP=5.03781;AC=4;AF=0.667;AN=6;AO=151;CIGAR=1X;DP=249;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.4161T&gt;C|ENSP00000368678.2:p.Thr1387%3D|synonymous_variant|LOW||acT/acC|T|4211|4161|Congenital_myasthenic_syndrome_8&amp;not_specified|Benign|criteria_provided&amp;_multiple_submitters&amp;_no_conflicts|4.3500e-01|||||||||-26|18|0|-2|0.00|0.00|0.00|0.00    GT:AD:AO:DP:PL:QA:QR:RO 1/1:1,67:67:68:1979,171,0:2234:34:1 0/1:52,35:35:87:812,0,1296:1190:1728:52 0/1:45,49:49:94:1219,0,1056:1666:1484:45
chr1    985266  .   C   T   2801.75 PASS    AB=0.491667;ABP=3.08268;AC=4;AF=0.667;AN=6;AO=110;CIGAR=1X;DP=171;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.4745-17C&gt;T||splice_polypyrimidine_tract_variant&amp;intron_variant|LOW||||||Congenital_myasthenic_syndrome_8&amp;not_specified|Benign|criteria_provided&amp;_multiple_submitters&amp;_no_conflicts|4.3500e-01|||||||||17|-4|25|-11|0.14|0.00|0.00|0.00  GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,51:51:51:1549,154,0:1718:0:0  0/1:33,27:27:60:649,0,825:918:1114:33   0/1:28,32:32:60:782,0,679:1066:952:28
chr1    986732  .   G   A   922.52  PASS    AB=0.415094;ABP=9.64763;AC=1;AF=0.167;AN=6;AO=44;CIGAR=1X;DP=356;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.5353G&gt;A|ENSP00000368678.2:p.Asp1785Asn|missense_variant|MODERATE||Gac/Aac|D/N|5403|5353|Congenital_myasthenic_syndrome_8&amp;not_specified&amp;not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&amp;_conflicting_interpretations|4.3500e-01|0.256992|-2.07|0.069&amp;.||1031|6.785660e-03|1|1.145360e-02|3|-37|3|17|0.00|0.00|0.00|0.00   GT:AD:AO:DP:PL:QA:QR:RO 0/0:129,0:0:129:0,388,3908:0:4340:129   0/1:62,44:44:106:1030,0,1581:1496:2108:62   0/0:121,0:0:121:0,364,3683:0:4090:121
chr1    987200  .   C   T   6412.21 PASS    AB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=219;CIGAR=1X;DP=219;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.5651+5C&gt;T||splice_donor_5th_base_variant&amp;intron_variant|LOW||||||Congenital_myasthenic_syndrome_8&amp;not_specified|Benign|criteria_provided&amp;_multiple_submitters&amp;_no_conflicts|4.3500e-01|||||||||-27|-5|-5|-29|0.00|0.00|0.00|0.00  GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,63:63:63:1911,190,0:2120:0:0  1/1:0,66:66:66:1963,199,0:2178:0:0  1/1:0,90:90:90:2737,271,0:3038:0:0
chr1    990280  .   C   T   4547.65 PASS    AB=0.446602;ABP=8.11221;AC=4;AF=0.667;AN=6;AO=175;CIGAR=1X;DP=289;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.6057C&gt;T|ENSP00000368678.2:p.Asp2019%3D|synonymous_variant|LOW||gaC/gaT|D|6107|6057|Congenital_myasthenic_syndrome_8&amp;not_specified|Benign|criteria_provided&amp;_multiple_submitters&amp;_no_conflicts|4.3500e-01|||||||||37|2|-23|-4|0.00|0.00|0.00|0.00    GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,83:83:83:2496,250,0:2770:0:0  0/1:56,39:39:95:910,0,1371:1326:1838:56 0/1:58,53:53:111:1283,0,1436:1794:1964:58
chr1    990380  .   C   T   570.29  PASS    AB=0.55102;ABP=4.11819;AC=2;AF=0.333;AN=6;AO=27;CIGAR=1X;DP=63;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.*19C&gt;T||3_prime_UTR_variant|MODIFIER||||6207||not_specified|Benign|criteria_provided&amp;_multiple_submitters&amp;_no_conflicts|4.3500e-01|||||||||7|-7|7|26|0.00|0.00|0.00|0.00   GT:AD:AO:DP:PL:QA:QR:RO 0/0:14,0:0:14:0,42,432:0:476:14 0/1:8,14:14:22:365,0,182:476:272:8  0/1:14,13:13:27:320,0,343:442:468:14
chr1    990417  .   T   C   328.36  PASS    AB=0.580645;ABP=4.76149;AC=2;AF=0.333;AN=6;AO=18;CIGAR=1X;DP=41;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.*56T&gt;C||3_prime_UTR_variant|MODIFIER||||6244||not_provided|Benign|criteria_provided&amp;_single_submitter|4.3500e-01|||||||||-44|29|8|-30|0.00|0.00|0.00|0.00 GT:AD:AO:DP:PL:QA:QR:RO 0/0:10,0:0:10:0,30,309:0:340:10 0/1:5,9:9:14:237,0,114:306:170:5    0/1:8,9:9:17:220,0,197:298:272:8
chr1    990517  .   C   T   182.67  SNP_filter  AB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=8;CIGAR=1X;DP=8;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.*156C&gt;T||3_prime_UTR_variant|MODIFIER||||6344||Congenital_myasthenic_syndrome_8&amp;not_provided|Benign|criteria_provided&amp;_multiple_submitters&amp;_no_conflicts|4.3500e-01|||||||||14|-28|-5|18|0.00|0.05|0.00|0.00  GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,1:1:1:34,3,0:34:0:0   1/1:0,4:4:4:126,12,0:136:0:0    1/1:0,3:3:3:95,9,0:102:0:0
chr1    990984  .   G   A   82.39   LowCoverage;SNP_filter  AB=0;ABP=0;AC=2;AF=1.00;AN=2;AO=3;CIGAR=1X;DP=3;CSQ=ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.*623G&gt;A||3_prime_UTR_variant|MODIFIER||||6811|||||4.3500e-01|||||||||5|-25|-42|5|0.00|0.00|0.00|0.00  GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,3:3:3:95,9,0:102:0:0  .   .
chr1    1007203 .   A   G   3110.06 PASS    AB=0;ABP=0;AC=6;AF=1.00;AN=6;AO=105;CIGAR=1X;DP=105;CSQ=ENSG00000237330|ENST00000453464|||ENST00000453464.2:c.744T&gt;C|ENSP00000410533.1:p.Asp248%3D|synonymous_variant|LOW||gaT/gaC|D|1045|744||||1.8670e+00|||||||||-7|-46|-26|2|0.00|0.00|0.01|0.00    GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,26:26:26:752,78,0:832:0:0 1/1:0,46:46:46:1411,138,0:1564:0:0  1/1:0,33:33:33:1013,99,0:1122:0:0
chr1    1007222 .   G   T   1780.81 PASS    AB=0.685185;ABP=35.1803;AC=2;AF=0.333;AN=6;AO=74;CIGAR=1X;DP=143;CSQ=ENSG00000237330|ENST00000453464|||ENST00000453464.2:c.725C&gt;A|ENSP00000410533.1:p.Pro242His|missense_variant|MODERATE||cCc/cAc|P/H|1026|725||||1.8670e+00|1.523231|3.44|0.182||18137|1.192080e-01|1398|2.038880e-01|-26|-35|-45|16|0.00|0.00|0.00|0.00  GT:AD:AO:DP:PL:QA:QR:RO 0/0:35,0:0:35:0,105,1034:0:1146:35  0/1:16,40:40:56:1044,0,277:1344:492:16  0/1:18,34:34:52:887,0,397:1156:612:18
chr1    1007432 .   G   A   3295.23 PASS    AB=0.44375;ABP=7.40753;AC=4;AF=0.667;AN=6;AO=128;CIGAR=1X;DP=217;CSQ=ENSG00000237330|ENST00000453464|||ENST00000453464.2:c.515C&gt;T|ENSP00000410533.1:p.Ala172Val|missense_variant|MODERATE||gCg/gTg|A/V|816|515||||1.8670e+00|-0.654559|-7.35|0.132||75853|4.996380e-01|20418|7.826930e-01|48|16|28|44|0.00|0.00|0.00|0.00   GT:AD:AO:DP:PL:QA:QR:RO 1/1:0,57:57:57:1740,172,0:1930:0:0  0/1:51,36:36:87:823,0,1302:1202:1734:51 0/1:38,35:35:73:854,0,946:1190:1292:38
```
</code></pre></div></td></tr></table></div>
<h4 id="le-header">Le header</h4>
<p>Lorsqu'il est utilisé, VEP ajoute dans le header</p>
<ul>
<li>la version du script et des DB utilisées</li>
<li>le champs <code>##INFO=&lt;ID=CSQ[...]&gt;</code> précisant l'ordre des annotations ajoutées pour chaque transcrit</li>
<li>La définition de toutes les colonnes personnalisées utilisées</li>
<li>la ligne de commande utilisée pour lancer l'annotation (peut être incomplète ou obfusquée dans le cas d'un recours à un fichier <code>config</code>)</li>
</ul>
<h4 id="le-champ-csq">Le champ CSQ</h4>
<p>L'annotation même est ajoutée dans la colonne INFO via un champs "CSQ" (par défaut) dédié.</p>
<ul>
<li>Pour chaque transcrit relevé, les annotations sont insérées dans l'ordre spécifié par la ligne <code>##INFO=&lt;ID=CSQ[...]&gt;</code> du header et séparées par des <code>|</code>.  </li>
<li>Les transcrits quant à eux sont séparés par des virgules <code>,</code>  </li>
<li>Enfin, le CSQ se conclue par un point-virgule <code>;</code> ou une tabulation.</li>
</ul>
<h5 id="exemple-de-plusieurs-transcrits">Exemple de plusieurs transcrits</h5>
<p>Exemple d'une annotation sur plusieurs transcrits (le retour à la ligne a été ajouté pour une meilleure lisibilité)
<div class="highlight"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre><span></span><span class="normal">1</span>
<span class="normal">2</span>
<span class="normal">3</span>
<span class="normal">4</span>
<span class="normal">5</span>
<span class="normal">6</span>
<span class="normal">7</span>
<span class="normal">8</span>
<span class="normal">9</span></pre></div></td><td class="code"><div><pre><span></span><code>#CHROM  POS ID  REF ALT QUAL    FILTER  INFO    FORMAT  22-23121-A-02-00    22-23123-A-02-00    22-24543-A-01-00
chr1    986732  .   G   A   922.52  PASS    AB=0.415094;ABP=9.64763;AC=1;AF=0.167;AN=6;AO=44;CIGAR=1X;DP=356; \
    CSQ=AGRN|ENSG00000188157|ENST00000379370|||ENST00000379370.2:c.5353G&gt;A|ENSP00000368678.2:p.Asp1785Asn|missense_variant|MODERATE||Gac/Aac|D/N|5403|5353|Congenital_myasthenic_syndrome_8&amp;not_specified&amp;not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&amp;_conflicting_interpretations|4.3500e-01|0.256992|-2.07|0.069&amp;.||1031|6.785660e-03|1|1.145360e-02|3|-37|3|17|0.00|0.00|0.00|0.00|1
        ,RP11-54O7.14|ENSG00000242590|ENST00000418300|||||upstream_gene_variant|MODIFIER||||||Congenital_myasthenic_syndrome_8&amp;not_specified&amp;not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&amp;_conflicting_interpretations||||||||||||||||||
        ,AGRN|ENSG00000188157|ENST00000419249|||ENST00000419249.1:c.262G&gt;A|ENSP00000400771.1:p.Asp88Asn|missense_variant|MODERATE||Gac/Aac|D/N|260|262|Congenital_myasthenic_syndrome_8&amp;not_specified&amp;not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&amp;_conflicting_interpretations|6.7500e-01|0.256992|-2.07|0.069&amp;.||1031|6.785660e-03|1|1.145360e-02|3|-37|3|17|0.00|0.00|0.00|0.00|
        ,AGRN|ENSG00000188157|ENST00000461111|||||upstream_gene_variant|MODIFIER||||||Congenital_myasthenic_syndrome_8&amp;not_specified&amp;not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&amp;_conflicting_interpretations||||||||||3|-37|3|17|0.00|0.00|0.00|0.00|
        ,AGRN|ENSG00000188157|ENST00000466223|||||downstream_gene_variant|MODIFIER||||||Congenital_myasthenic_syndrome_8&amp;not_specified&amp;not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&amp;_conflicting_interpretations||||||||||3|-37|3|17|0.00|0.00|0.00|0.00|
        ,AGRN|ENSG00000188157|ENST00000478677|||||downstream_gene_variant|MODIFIER||||||Congenital_myasthenic_syndrome_8&amp;not_specified&amp;not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&amp;_conflicting_interpretations||||||||||3|-37|3|17|0.00|0.00|0.00|0.00|
        ,AGRN|ENSG00000188157|ENST00000492947|||||downstream_gene_variant|MODIFIER||||||Congenital_myasthenic_syndrome_8&amp;not_specified&amp;not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&amp;_conflicting_interpretations||||||||||3|-37|3|17|0.00|0.00|0.00|0.00|    GT:AD:AO:DP:PL:QA:QR:RO 0/0:129,0:0:129:0,388,3908:0:4340:129   0/1:62,44:44:106:1030,0,1581:1496:2108:62   0/0:121,0:0:121:0,364,3683:0:4090:121
</code></pre></div></td></tr></table></div>
Il est à noter que le deuxième transcrit annoté concerne un second gène (<em>upstream_gene_variant</em> signifiant que la variation est en amont de ce gène).</p>
<h5 id="exemple-de-plusieurs-alt">Exemple de plusieurs ALT</h5>
<p>Dans le cas d'un VCF non normalisé, plusieurs ALT sont notées sur une seule ligne du VCF. VEP fera l'annotation pour toutes les ALT au sein du même champ CSQ et les séparera par des virgules. Il reste possible de les différencier grâce à la colonne <code>Allele</code> permettant de savoir quelle ALT est concernée par l'annotation jusqu'à la virgule suivante.
<div class="highlight"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre><span></span><span class="normal"> 1</span>
<span class="normal"> 2</span>
<span class="normal"> 3</span>
<span class="normal"> 4</span>
<span class="normal"> 5</span>
<span class="normal"> 6</span>
<span class="normal"> 7</span>
<span class="normal"> 8</span>
<span class="normal"> 9</span>
<span class="normal">10</span>
<span class="normal">11</span>
<span class="normal">12</span></pre></div></td><td class="code"><div><pre><span></span><code>// CSQ = Allele|Consequence|Impact|Symbol|Gene|Feature_type|Feature|…
// Pour les variations : 
#CHROM  POS ID  REF ALT
chr1    889455  .   G   A, T [...]
CSQ=A|stop_gained|HIGH|NOC2L|ENSG00000188976|transcript|ENST00000327044|protein_coding|7/19|c.706C&gt;T|p.Gln236*|756/2790|706/2250|236/749||
    ,T|missense_variant|MODERATE|NOC2L|ENSG00000188976|transcript|ENST00000327044|protein_coding|7/19|c.706C&gt;A|p.Gln236Lys|756/2790|706/2250|236/749||
    ,A|downstream_gene_variant|MODIFIER|NOC2L|ENSG00000188976|transcript|ENST00000487214|processed_transcript||n.*865C&gt;T|||||351|
    ,T|downstream_gene_variant|MODIFIER|NOC2L|ENSG00000188976|transcript|ENST00000487214|processed_transcript||n.*865C&gt;A|||||351|
    ,A|downstream_gene_variant|MODIFIER|NOC2L|ENSG00000188976|transcript|ENST00000469563|retained_intron||n.*878C&gt;T|||||4171|
    ,T|downstream_gene_variant|MODIFIER|NOC2L|ENSG00000188976|transcript|ENST00000469563|retained_intron||n.*878C&gt;A|||||4171|
    ,A|non_coding_exon_variant|MODIFIER|NOC2L|ENSG00000188976|transcript|ENST00000477976|retained_intron|5/17|n.2153C&gt;T||||||
    ,T|non_coding_exon_variant|MODIFIER|NOC2L|ENSG00000188976|transcript|ENST00000477976|retained_intron|5/17|n.2153C&gt;A||||||;
</code></pre></div></td></tr></table></div></p>
<h5 id="reformatage-par-vep">Reformatage par VEP</h5>
<p>Si une annotation contient une virgule ou un pipe, VEP la remplace par une esperluette <code>&amp;</code></p>
<p>Par exemple :
<div class="highlight"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre><span></span><span class="normal">1</span>
<span class="normal">2</span>
<span class="normal">3</span>
<span class="normal">4</span></pre></div></td><td class="code"><div><pre><span></span><code>// Dans le VCF de ClinVar
[...]CLNDN=Congenital_myasthenic_syndrome_8|not_specified|not_provided;CLNHGVS=NC_000001.10:g.986732G&gt;A;CLNREVSTAT=criteria_provided,_conflicting_interpretations;CLNSIG=Conflicting_interpretations_of_pathogenicity[...]
// Dans le VCF annoté
[...]|Congenital_myasthenic_syndrome_8&amp;not_specified&amp;not_provided|Conflicting_interpretations_of_pathogenicity|criteria_provided&amp;_conflicting_interpretations|[...]
</code></pre></div></td></tr></table></div></p>

  <hr>
<div class="md-source-file">
  <small>
    
      Last update:
      <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date">September 28, 2023</span>
      
    
  </small>
</div>





                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      2020 - 2021 © DiaghoProject
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        <div class="md-social">
  
    
    
    
    
    <a href="https://github.com/DiaghoProject" target="_blank" rel="noopener" title="DiaghoProject on GitHub" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "../../..", "features": ["navigation.indexes"], "search": "../../../assets/javascripts/workers/search.a264c092.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../../../assets/javascripts/bundle.726fbb30.min.js"></script>
      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/tablesort/5.2.1/tablesort.min.js"></script>
      
        <script src="../../../javascripts/tables.js"></script>
      
    
  </body>
</html>